<?php

	include '../../../../common/include/common.php';
    include_once '../../../config.php';
    $module = modules::GATK;
    $name = docSN::toolDocs;

    printHeader($module, $name, topSN::guide);

    $selected_major_version = 4;
    $dirs = lookupVersionDirs($selected_major_version);
?>

<link type='text/css' rel='stylesheet' href='gatkDoc.css'>

<div class="row hide_me_html" style="margin: 14px 3px 14px 3px;" id="tippy-top">

	<i class='<?php print ico::toolDocsIcon; ?>'></i> <em>Showing tool doc from version 4.2.4.0 | The latest version is <?php print $latest_version; ?></em>
	<div class="btn-group pull-right">
		<a class="btn btn-warning dropdown-toggle" data-toggle="dropdown" href="#">
			4.2.4.0
			<span class="caret"></span>
		</a>
		<ul class="dropdown-menu">
			<?php foreach($dirs as $dir) { ?>
			<li class="hide_me_html"><a tabindex='-1' href='../<?=$dir?>' ><?=$dir?></a></li>
			<?php } ?>
		</ul>
	</div>
	<hr />

</div>


<div class='row' id="top">



	<?php $group = 'Variant Evaluation and Refinement'; ?>

	<section class="col-md-4">
		<aside class="well">
			<a class="hide_me_php" href="index.html"><h4><i class='fa fa-chevron-left'></i> Back to Tool Documentation Index</h4></a>
			<a class="hide_me_html" href=".."><h4><i class='fa fa-chevron-left'></i> Back to Tool Documentation Index</h4></a>
		</aside>
		<aside class="well hide_me_html">
			<h3>Browse tools by category</h3>

        <style>
            #sidenav .accordion-body a {
                color : gray;
            }

            .accordion-body li {
                list-style : none;
            }
        </style>

        <ul class="nav nav-pills nav-stacked" id="sidenav">
        		<hr>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#CopyNumberVariantDiscovery">Copy Number Variant Discovery</a>
							<div id="CopyNumberVariantDiscovery"
								<?php echo ($group == 'Copy Number Variant Discovery')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_AnnotateIntervals.html">AnnotateIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_CallCopyRatioSegments.html">CallCopyRatioSegments</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_CreateReadCountPanelOfNormals.html">CreateReadCountPanelOfNormals</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_DenoiseReadCounts.html">DenoiseReadCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_DetermineGermlineContigPloidy.html">DetermineGermlineContigPloidy</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_FilterIntervals.html">FilterIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_GermlineCNVCaller.html">GermlineCNVCaller</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_ModelSegments.html">ModelSegments</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_plotting_PlotDenoisedCopyRatios.html">PlotDenoisedCopyRatios</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_plotting_PlotModeledSegments.html">PlotModeledSegments</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_PostprocessGermlineCNVCalls.html">PostprocessGermlineCNVCalls</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#CoverageAnalysis">Coverage Analysis</a>
							<div id="CoverageAnalysis"
								<?php echo ($group == 'Coverage Analysis')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_rnaseq_ASEReadCounter.html">ASEReadCounter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_AnalyzeSaturationMutagenesis.html">AnalyzeSaturationMutagenesis</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_CollectAllelicCounts.html">CollectAllelicCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_CollectReadCounts.html">CollectReadCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_CountBases.html">CountBases</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_CountBasesSpark.html">CountBasesSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_CountReads.html">CountReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_CountReadsSpark.html">CountReadsSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_coverage_DepthOfCoverage.html">DepthOfCoverage</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_rnaseq_GeneExpressionEvaluation.html">GeneExpressionEvaluation</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_contamination_GetPileupSummaries.html">GetPileupSummaries</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_LocalAssembler.html">LocalAssembler</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_qc_Pileup.html">Pileup</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_PileupSpark.html">PileupSpark</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#DiagnosticsandQualityControl">Diagnostics and Quality Control</a>
							<div id="DiagnosticsandQualityControl"
								<?php echo ($group == 'Diagnostics and Quality Control')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_vcf_AccumulateVariantCallingMetrics.html">AccumulateVariantCallingMetrics (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_bqsr_AnalyzeCovariates.html">AnalyzeCovariates</a>
											</li>
											<li>
												<a href="picard_sam_BamIndexStats.html">BamIndexStats (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_evidence_CalcMetadataSpark.html">CalcMetadataSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_contamination_CalculateContamination.html">CalculateContamination</a>
											</li>
											<li>
												<a href="picard_fingerprint_CalculateFingerprintMetrics.html">CalculateFingerprintMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_CalculateReadGroupChecksum.html">CalculateReadGroupChecksum (Picard)</a>
											</li>
											<li>
												<a href="picard_fingerprint_CheckFingerprint.html">CheckFingerprint (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_qc_CheckPileup.html">CheckPileup</a>
											</li>
											<li>
												<a href="picard_sam_CheckTerminatorBlock.html">CheckTerminatorBlock (Picard)</a>
											</li>
											<li>
												<a href="picard_fingerprint_ClusterCrosscheckMetrics.html">ClusterCrosscheckMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectAlignmentSummaryMetrics.html">CollectAlignmentSummaryMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_arrays_CollectArraysVariantCallingMetrics.html">CollectArraysVariantCallingMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectBaseDistributionByCycle.html">CollectBaseDistributionByCycle (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectBaseDistributionByCycleSpark.html">CollectBaseDistributionByCycleSpark</a>
											</li>
											<li>
												<a href="picard_analysis_CollectGcBiasMetrics.html">CollectGcBiasMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_illumina_quality_CollectHiSeqXPfFailMetrics.html">CollectHiSeqXPfFailMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_directed_CollectHsMetrics.html">CollectHsMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_replicates_CollectIndependentReplicateMetrics.html">CollectIndependentReplicateMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectInsertSizeMetrics.html">CollectInsertSizeMetrics (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectInsertSizeMetricsSpark.html">CollectInsertSizeMetricsSpark</a>
											</li>
											<li>
												<a href="picard_analysis_CollectJumpingLibraryMetrics.html">CollectJumpingLibraryMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectMultipleMetrics.html">CollectMultipleMetrics (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectMultipleMetricsSpark.html">CollectMultipleMetricsSpark</a>
											</li>
											<li>
												<a href="picard_analysis_CollectOxoGMetrics.html">CollectOxoGMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectQualityYieldMetrics.html">CollectQualityYieldMetrics (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectQualityYieldMetricsSpark.html">CollectQualityYieldMetricsSpark</a>
											</li>
											<li>
												<a href="picard_analysis_CollectRawWgsMetrics.html">CollectRawWgsMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectRnaSeqMetrics.html">CollectRnaSeqMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectRrbsMetrics.html">CollectRrbsMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_SamErrorMetric_CollectSamErrorMetrics.html">CollectSamErrorMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_artifacts_CollectSequencingArtifactMetrics.html">CollectSequencingArtifactMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_directed_CollectTargetedPcrMetrics.html">CollectTargetedPcrMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_vcf_CollectVariantCallingMetrics.html">CollectVariantCallingMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectWgsMetrics.html">CollectWgsMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_CollectWgsMetricsWithNonZeroCoverage.html">CollectWgsMetricsWithNonZeroCoverage (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_validation_CompareBaseQualities.html">CompareBaseQualities</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_validation_CompareDuplicatesSpark.html">CompareDuplicatesSpark</a>
											</li>
											<li>
												<a href="picard_analysis_CompareMetrics.html">CompareMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_CompareSAMs.html">CompareSAMs (Picard)</a>
											</li>
											<li>
												<a href="picard_analysis_artifacts_ConvertSequencingArtifactToOxoG.html">ConvertSequencingArtifactToOxoG (Picard)</a>
											</li>
											<li>
												<a href="picard_fingerprint_CrosscheckFingerprints.html">CrosscheckFingerprints (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_markduplicates_EstimateLibraryComplexity.html">EstimateLibraryComplexity (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_FlagStat.html">FlagStat</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_FlagStatSpark.html">FlagStatSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_GetSampleName.html">GetSampleName</a>
											</li>
											<li>
												<a href="picard_analysis_MeanQualityByCycle.html">MeanQualityByCycle (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_metrics_MeanQualityByCycleSpark.html">MeanQualityByCycleSpark</a>
											</li>
											<li>
												<a href="picard_analysis_QualityScoreDistribution.html">QualityScoreDistribution (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_metrics_QualityScoreDistributionSpark.html">QualityScoreDistributionSpark</a>
											</li>
											<li>
												<a href="picard_sam_ValidateSamFile.html">ValidateSamFile (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_ViewSam.html">ViewSam (Picard)</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#IntervalsManipulation">Intervals Manipulation</a>
							<div id="IntervalsManipulation"
								<?php echo ($group == 'Intervals Manipulation')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_util_BedToIntervalList.html">BedToIntervalList (Picard)</a>
											</li>
											<li>
												<a href="picard_util_IntervalListToBed.html">IntervalListToBed (Picard)</a>
											</li>
											<li>
												<a href="picard_util_IntervalListTools.html">IntervalListTools (Picard)</a>
											</li>
											<li>
												<a href="picard_util_LiftOverIntervalList.html">LiftOverIntervalList (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_copynumber_PreprocessIntervals.html">PreprocessIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_SplitIntervals.html">SplitIntervals</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#Metagenomics">Metagenomics</a>
							<div id="Metagenomics"
								<?php echo ($group == 'Metagenomics')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildKmers.html">PathSeqBuildKmers</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildReferenceTaxonomy.html">PathSeqBuildReferenceTaxonomy</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBwaSpark.html">PathSeqBwaSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqFilterSpark.html">PathSeqFilterSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqPipelineSpark.html">PathSeqPipelineSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqScoreSpark.html">PathSeqScoreSpark</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#Other">Other</a>
							<div id="Other"
								<?php echo ($group == 'Other')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_CreateHadoopBamSplittingIndex.html">CreateHadoopBamSplittingIndex</a>
											</li>
											<li>
												<a href="picard_util_FifoBuffer.html">FifoBuffer (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_bqsr_GatherBQSRReports.html">GatherBQSRReports</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_vqsr_GatherTranches.html">GatherTranches</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_IndexFeatureFile.html">IndexFeatureFile</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_ParallelCopyGCSDirectoryIntoHDFSSpark.html">ParallelCopyGCSDirectoryIntoHDFSSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_ReadAnonymizer.html">ReadAnonymizer</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_variantutils_ReblockGVCF.html">ReblockGVCF</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ReadDataManipulation">Read Data Manipulation</a>
							<div id="ReadDataManipulation"
								<?php echo ($group == 'Read Data Manipulation')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_sam_AddCommentsToBam.html">AddCommentsToBam (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_AddOATag.html">AddOATag (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_AddOrReplaceReadGroups.html">AddOrReplaceReadGroups (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_bqsr_ApplyBQSR.html">ApplyBQSR</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_ApplyBQSRSpark.html">ApplyBQSRSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_BQSRPipelineSpark.html">BQSRPipelineSpark</a>
											</li>
											<li>
												<a href="picard_fastq_BamToBfq.html">BamToBfq (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_bqsr_BaseRecalibrator.html">BaseRecalibrator</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_BaseRecalibratorSpark.html">BaseRecalibratorSpark</a>
											</li>
											<li>
												<a href="picard_sam_BuildBamIndex.html">BuildBamIndex (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_BwaAndMarkDuplicatesPipelineSpark.html">BwaAndMarkDuplicatesPipelineSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_bwa_BwaSpark.html">BwaSpark</a>
											</li>
											<li>
												<a href="picard_sam_CleanSam.html">CleanSam (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_ClipReads.html">ClipReads</a>
											</li>
											<li>
												<a href="picard_sam_markduplicates_CollectDuplicateMetrics.html">CollectDuplicateMetrics (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_ConvertHeaderlessHadoopBamShardToBam.html">ConvertHeaderlessHadoopBamShardToBam</a>
											</li>
											<li>
												<a href="picard_sam_DownsampleSam.html">DownsampleSam (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_utils_ExtractOriginalAlignmentRecordsByNameSpark.html">ExtractOriginalAlignmentRecordsByNameSpark</a>
											</li>
											<li>
												<a href="picard_sam_FastqToSam.html">FastqToSam (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_FilterSamReads.html">FilterSamReads (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_FixMateInformation.html">FixMateInformation (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_FixMisencodedBaseQualityReads.html">FixMisencodedBaseQualityReads</a>
											</li>
											<li>
												<a href="picard_sam_GatherBamFiles.html">GatherBamFiles (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_LeftAlignIndels.html">LeftAlignIndels</a>
											</li>
											<li>
												<a href="picard_sam_markduplicates_MarkDuplicates.html">MarkDuplicates (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.html">MarkDuplicatesSpark</a>
											</li>
											<li>
												<a href="picard_sam_markduplicates_MarkDuplicatesWithMateCigar.html">MarkDuplicatesWithMateCigar (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_MergeBamAlignment.html">MergeBamAlignment (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_MergeSamFiles.html">MergeSamFiles (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_PositionBasedDownsampleSam.html">PositionBasedDownsampleSam (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_PrintDistantMates.html">PrintDistantMates</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_PrintReads.html">PrintReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_PrintReadsHeader.html">PrintReadsHeader</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_PrintReadsSpark.html">PrintReadsSpark</a>
											</li>
											<li>
												<a href="picard_sam_ReorderSam.html">ReorderSam (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_ReplaceSamHeader.html">ReplaceSamHeader (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_RevertBaseQualityScores.html">RevertBaseQualityScores</a>
											</li>
											<li>
												<a href="picard_sam_RevertOriginalBaseQualitiesAndAddMateCigar.html">RevertOriginalBaseQualitiesAndAddMateCigar (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_RevertSam.html">RevertSam (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_RevertSamSpark.html">RevertSamSpark</a>
											</li>
											<li>
												<a href="picard_sam_SamFormatConverter.html">SamFormatConverter (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_SamToFastq.html">SamToFastq (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_SetNmAndUqTags.html">SetNmAndUqTags (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_SetNmMdAndUqTags.html">SetNmMdAndUqTags (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_markduplicates_SimpleMarkDuplicatesWithMateCigar.html">SimpleMarkDuplicatesWithMateCigar (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_SortSam.html">SortSam (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_SortSamSpark.html">SortSamSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_rnaseq_SplitNCigarReads.html">SplitNCigarReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_SplitReads.html">SplitReads</a>
											</li>
											<li>
												<a href="picard_sam_SplitSamByLibrary.html">SplitSamByLibrary (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_SplitSamByNumberOfReads.html">SplitSamByNumberOfReads (Picard)</a>
											</li>
											<li>
												<a href="picard_sam_markduplicates_UmiAwareMarkDuplicatesWithMateCigar.html">UmiAwareMarkDuplicatesWithMateCigar (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_UnmarkDuplicates.html">UnmarkDuplicates</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#Reference">Reference</a>
							<div id="Reference"
								<?php echo ($group == 'Reference')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_util_BaitDesigner.html">BaitDesigner (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_BwaMemIndexImageCreator.html">BwaMemIndexImageCreator</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_dragstr_ComposeSTRTableFile.html">ComposeSTRTableFile</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_fasta_CountBasesInReference.html">CountBasesInReference</a>
											</li>
											<li>
												<a href="picard_sam_CreateSequenceDictionary.html">CreateSequenceDictionary (Picard)</a>
											</li>
											<li>
												<a href="picard_reference_ExtractSequences.html">ExtractSequences (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_fasta_FastaAlternateReferenceMaker.html">FastaAlternateReferenceMaker</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_fasta_FastaReferenceMaker.html">FastaReferenceMaker</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBadGenomicKmersSpark.html">FindBadGenomicKmersSpark</a>
											</li>
											<li>
												<a href="picard_reference_NonNFastaSize.html">NonNFastaSize (Picard)</a>
											</li>
											<li>
												<a href="picard_reference_NormalizeFasta.html">NormalizeFasta (Picard)</a>
											</li>
											<li>
												<a href="picard_util_ScatterIntervalsByNs.html">ScatterIntervalsByNs (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_fasta_ShiftFasta.html">ShiftFasta</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ShortVariantDiscovery">Short Variant Discovery</a>
							<div id="ShortVariantDiscovery"
								<?php echo ($group == 'Short Variant Discovery')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_dragstr_CalibrateDragstrModel.html">CalibrateDragstrModel</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_CombineGVCFs.html">CombineGVCFs</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_genomicsdb_GenomicsDBImport.html">GenomicsDBImport</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_GenotypeGVCFs.html">GenotypeGVCFs</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_gnarlyGenotyper_GnarlyGenotyper.html">GnarlyGenotyper</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_haplotypecaller_HaplotypeCaller.html">HaplotypeCaller</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_HaplotypeCallerSpark.html">HaplotypeCallerSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_readorientation_LearnReadOrientationModel.html">LearnReadOrientationModel</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.html">Mutect2</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_ReadsPipelineSpark.html">ReadsPipelineSpark</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#StructuralVariantDiscovery">Structural Variant Discovery</a>
							<div id="StructuralVariantDiscovery"
								<?php echo ($group == 'Structural Variant Discovery')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_discovery_inference_CpxVariantReInterpreterSpark.html">CpxVariantReInterpreterSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_discovery_DiscoverVariantsFromContigAlignmentsSAMSpark.html">DiscoverVariantsFromContigAlignmentsSAMSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_evidence_ExtractSVEvidenceSpark.html">ExtractSVEvidenceSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBreakpointEvidenceSpark.html">FindBreakpointEvidenceSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_sv_PrintSVEvidence.html">PrintSVEvidence</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_StructuralVariationDiscoveryPipelineSpark.html">StructuralVariationDiscoveryPipelineSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_sv_discovery_SvDiscoverFromLocalAssemblyContigAlignmentsSpark.html">SvDiscoverFromLocalAssemblyContigAlignmentsSpark</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantEvaluationandRefinement">Variant Evaluation and Refinement</a>
							<div id="VariantEvaluationandRefinement"
								<?php echo ($group == 'Variant Evaluation and Refinement')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithBamDepth.html">AnnotateVcfWithBamDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithExpectedAlleleFraction.html">AnnotateVcfWithExpectedAlleleFraction</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_variantutils_CalculateGenotypePosteriors.html">CalculateGenotypePosteriors</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_validation_CalculateMixingFractions.html">CalculateMixingFractions</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_validation_Concordance.html">Concordance</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_validation_CountFalsePositives.html">CountFalsePositives</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_CountVariants.html">CountVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_CountVariantsSpark.html">CountVariantsSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_validation_EvaluateInfoFieldConcordance.html">EvaluateInfoFieldConcordance</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_funcotator_FilterFuncotations.html">FilterFuncotations</a>
											</li>
											<li>
												<a href="picard_vcf_MendelianViolations_FindMendelianViolations.html">FindMendelianViolations (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_funcotator_FuncotateSegments.html">FuncotateSegments</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_funcotator_Funcotator.html">Funcotator</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_funcotator_FuncotatorDataSourceDownloader.html">FuncotatorDataSourceDownloader</a>
											</li>
											<li>
												<a href="picard_vcf_GenotypeConcordance.html">GenotypeConcordance (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_validation_basicshortmutpileup_ValidateBasicSomaticShortMutations.html">ValidateBasicSomaticShortMutations</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_variantutils_ValidateVariants.html">ValidateVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_varianteval_VariantEval.html">VariantEval</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_variantutils_VariantsToTable.html">VariantsToTable</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantFiltering">Variant Filtering</a>
							<div id="VariantFiltering"
								<?php echo ($group == 'Variant Filtering')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_vqsr_ApplyVQSR.html">ApplyVQSR</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_vqsr_CNNScoreVariants.html">CNNScoreVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantTrain.html">CNNVariantTrain</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantWriteTensors.html">CNNVariantWriteTensors</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.html">CreateSomaticPanelOfNormals</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_realignmentfilter_FilterAlignmentArtifacts.html">FilterAlignmentArtifacts</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_mutect_filtering_FilterMutectCalls.html">FilterMutectCalls</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_vqsr_FilterVariantTranches.html">FilterVariantTranches</a>
											</li>
											<li>
												<a href="picard_vcf_filter_FilterVcf.html">FilterVcf (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_filters_VariantFiltration.html">VariantFiltration</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_vqsr_VariantRecalibrator.html">VariantRecalibrator</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantManipulation">Variant Manipulation</a>
							<div id="VariantManipulation"
								<?php echo ($group == 'Variant Manipulation')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_vcf_FixVcfHeader.html">FixVcfHeader (Picard)</a>
											</li>
											<li>
												<a href="picard_vcf_GatherVcfs.html">GatherVcfs (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_GatherVcfsCloud.html">GatherVcfsCloud</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_variantutils_LeftAlignAndTrimVariants.html">LeftAlignAndTrimVariants</a>
											</li>
											<li>
												<a href="picard_vcf_LiftoverVcf.html">LiftoverVcf (Picard)</a>
											</li>
											<li>
												<a href="picard_vcf_MakeSitesOnlyVcf.html">MakeSitesOnlyVcf (Picard)</a>
											</li>
											<li>
												<a href="picard_vcf_MergeVcfs.html">MergeVcfs (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_spark_pipelines_PrintVariantsSpark.html">PrintVariantsSpark</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_validation_RemoveNearbyIndels.html">RemoveNearbyIndels</a>
											</li>
											<li>
												<a href="picard_vcf_RenameSampleInVcf.html">RenameSampleInVcf (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_variantutils_SelectVariants.html">SelectVariants</a>
											</li>
											<li>
												<a href="picard_vcf_SortVcf.html">SortVcf (Picard)</a>
											</li>
											<li>
												<a href="picard_vcf_SplitVcfs.html">SplitVcfs (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_variantutils_UpdateVCFSequenceDictionary.html">UpdateVCFSequenceDictionary</a>
											</li>
											<li>
												<a href="picard_vcf_UpdateVcfSequenceDictionary.html">UpdateVcfSequenceDictionary (Picard)</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_VariantAnnotator.html">VariantAnnotator</a>
											</li>
											<li>
												<a href="picard_vcf_VcfFormatConverter.html">VcfFormatConverter (Picard)</a>
											</li>
											<li>
												<a href="picard_vcf_VcfToIntervalList.html">VcfToIntervalList (Picard)</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#BaseCalling">Base Calling</a>
							<div id="BaseCalling"
								<?php echo ($group == 'Base Calling')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_illumina_CheckIlluminaDirectory.html">CheckIlluminaDirectory (Picard)</a>
											</li>
											<li>
												<a href="picard_illumina_CollectIlluminaBasecallingMetrics.html">CollectIlluminaBasecallingMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_illumina_CollectIlluminaLaneMetrics.html">CollectIlluminaLaneMetrics (Picard)</a>
											</li>
											<li>
												<a href="picard_illumina_ExtractIlluminaBarcodes.html">ExtractIlluminaBarcodes (Picard)</a>
											</li>
											<li>
												<a href="picard_illumina_IlluminaBasecallsToFastq.html">IlluminaBasecallsToFastq (Picard)</a>
											</li>
											<li>
												<a href="picard_illumina_IlluminaBasecallsToSam.html">IlluminaBasecallsToSam (Picard)</a>
											</li>
											<li>
												<a href="picard_illumina_MarkIlluminaAdapters.html">MarkIlluminaAdapters (Picard)</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#GenotypingArraysManipulation">Genotyping Arrays Manipulation</a>
							<div id="GenotypingArraysManipulation"
								<?php echo ($group == 'Genotyping Arrays Manipulation')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_arrays_illumina_BpmToNormalizationManifestCsv.html">BpmToNormalizationManifestCsv (Picard)</a>
											</li>
											<li>
												<a href="picard_arrays_CombineGenotypingArrayVcfs.html">CombineGenotypingArrayVcfs (Picard)</a>
											</li>
											<li>
												<a href="picard_arrays_CreateBafRegressMetricsFile.html">CreateBafRegressMetricsFile (Picard)</a>
											</li>
											<li>
												<a href="picard_arrays_CreateVerifyIDIntensityContaminationMetricsFile.html">CreateVerifyIDIntensityContaminationMetricsFile (Picard)</a>
											</li>
											<li>
												<a href="picard_arrays_GtcToVcf.html">GtcToVcf (Picard)</a>
											</li>
											<li>
												<a href="picard_arrays_MergePedIntoVcf.html">MergePedIntoVcf (Picard)</a>
											</li>
											<li>
												<a href="picard_arrays_VcfToAdpc.html">VcfToAdpc (Picard)</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#MethylationSpecificTools">Methylation-Specific Tools</a>
							<div id="MethylationSpecificTools"
								<?php echo ($group == 'Methylation-Specific Tools')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_MethylationTypeCaller.html">MethylationTypeCaller</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ReadFilters">Read Filters</a>
							<div id="ReadFilters"
								<?php echo ($group == 'Read Filters')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_AlignmentAgreesWithHeaderReadFilter.html">AlignmentAgreesWithHeaderReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$AllowAllReadsReadFilter.html">AllowAllReadsReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_AmbiguousBaseReadFilter.html">AmbiguousBaseReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$CigarContainsNoNOperator.html">CigarContainsNoNOperator</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$FirstOfPairReadFilter.html">FirstOfPairReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_FragmentLengthReadFilter.html">FragmentLengthReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$GoodCigarReadFilter.html">GoodCigarReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$HasReadGroupReadFilter.html">HasReadGroupReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_IntervalOverlapReadFilter.html">IntervalOverlapReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_LibraryReadFilter.html">LibraryReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappedReadFilter.html">MappedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityAvailableReadFilter.html">MappingQualityAvailableReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityNotZeroReadFilter.html">MappingQualityNotZeroReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_MappingQualityReadFilter.html">MappingQualityReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MatchingBasesAndQualsReadFilter.html">MatchingBasesAndQualsReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateDifferentStrandReadFilter.html">MateDifferentStrandReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_MateDistantReadFilter.html">MateDistantReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateOnSameContigOrNoMappedMateReadFilter.html">MateOnSameContigOrNoMappedMateReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateUnmappedAndUnmappedReadFilter.html">MateUnmappedAndUnmappedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_MetricsReadFilter.html">MetricsReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonChimericOriginalAlignmentReadFilter.html">NonChimericOriginalAlignmentReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroFragmentLengthReadFilter.html">NonZeroFragmentLengthReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroReferenceLengthAlignmentReadFilter.html">NonZeroReferenceLengthAlignmentReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotDuplicateReadFilter.html">NotDuplicateReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotProperlyPairedReadFilter.html">NotProperlyPairedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSecondaryAlignmentReadFilter.html">NotSecondaryAlignmentReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSupplementaryAlignmentReadFilter.html">NotSupplementaryAlignmentReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_OverclippedReadFilter.html">OverclippedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PairedReadFilter.html">PairedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PassesVendorQualityCheckReadFilter.html">PassesVendorQualityCheckReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_PlatformReadFilter.html">PlatformReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_PlatformUnitReadFilter.html">PlatformUnitReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PrimaryLineReadFilter.html">PrimaryLineReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ProperlyPairedReadFilter.html">ProperlyPairedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadGroupBlackListReadFilter.html">ReadGroupBlackListReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadGroupReadFilter.html">ReadGroupReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ReadLengthEqualsCigarLengthReadFilter.html">ReadLengthEqualsCigarLengthReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadLengthReadFilter.html">ReadLengthReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadNameReadFilter.html">ReadNameReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadStrandFilter.html">ReadStrandFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_SampleReadFilter.html">SampleReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SecondOfPairReadFilter.html">SecondOfPairReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SeqIsStoredReadFilter.html">SeqIsStoredReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_SoftClippedReadFilter.html">SoftClippedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentEndReadFilter.html">ValidAlignmentEndReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentStartReadFilter.html">ValidAlignmentStartReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_engine_filters_WellformedReadFilter.html">WellformedReadFilter</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantAnnotations">Variant Annotations</a>
							<div id="VariantAnnotations"
								<?php echo ($group == 'Variant Annotations')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_BaseQualityRankSumTest.html">AS_BaseQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_FisherStrand.html">AS_FisherStrand</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_InbreedingCoeff.html">AS_InbreedingCoeff</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_MappingQualityRankSumTest.html">AS_MappingQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_QualByDepth.html">AS_QualByDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_RMSMappingQuality.html">AS_RMSMappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_ReadPosRankSumTest.html">AS_ReadPosRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_StrandOddsRatio.html">AS_StrandOddsRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_AlleleFraction.html">AlleleFraction</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_AllelePseudoDepth.html">AllelePseudoDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_AssemblyComplexity.html">AssemblyComplexity</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_BaseQuality.html">BaseQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_BaseQualityRankSumTest.html">BaseQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_ChromosomeCounts.html">ChromosomeCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_ClippingRankSumTest.html">ClippingRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_CountNs.html">CountNs</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_Coverage.html">Coverage</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerAlleleBySample.html">DepthPerAlleleBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerSampleHC.html">DepthPerSampleHC</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_ExcessHet.html">ExcessHet</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_FeaturizedReadSets.html">FeaturizedReadSets</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.html">FisherStrand</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_FragmentDepthPerAlleleBySample.html">FragmentDepthPerAlleleBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_FragmentLength.html">FragmentLength</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_GenotypeSummaries.html">GenotypeSummaries</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_InbreedingCoeff.html">InbreedingCoeff</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_LikelihoodRankSumTest.html">LikelihoodRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_MappingQuality.html">MappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityRankSumTest.html">MappingQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityZero.html">MappingQualityZero</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_OrientationBiasReadCounts.html">OrientationBiasReadCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_OriginalAlignment.html">OriginalAlignment</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_PossibleDeNovo.html">PossibleDeNovo</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_QualByDepth.html">QualByDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_RMSMappingQuality.html">RMSMappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosRankSumTest.html">ReadPosRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosition.html">ReadPosition</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_ReferenceBases.html">ReferenceBases</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_SampleList.html">SampleList</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_StrandBiasBySample.html">StrandBiasBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_StrandOddsRatio.html">StrandOddsRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_TandemRepeat.html">TandemRepeat</a>
											</li>
											<li>
												<a href="org_broadinstitute_hellbender_tools_walkers_annotator_UniqueAltReadCount.html">UniqueAltReadCount</a>
											</li>
								</ul>
							</div>
						</li>
        </ul>
		</aside>

	</section>

	<div class="col-md-8">

			<h1>Funcotator</h1>

			<p>Functional Annotator</p>

			<h3>Category
				<small> Variant Evaluation and Refinement</small>
			</h3>
		<hr>
		<h2>Overview</h2>
		Funcotator (FUNCtional annOTATOR) analyzes given variants for their function (as retrieved from a set of data sources) and produces the analysis in a specified output file.

 <p>
     This tool is a functional annotation tool that allows a user to add annotations to called variants based on a set of data sources, each with its own matching criteria.
 </p>

 <h3>Detailed Information and Tutorial</h3>
 <p>Detailed information and a tutorial can be found here:
     <ul>
         <li><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360035889931-Funcotator-Information-and-Tutorial">https://gatk.broadinstitute.org/hc/en-us/articles/360035889931-Funcotator-Information-and-Tutorial</a></li>
     </ul>
 </p>

 <h3>Data Sources</h3>
 <p>
     Data sources are expected to be in folders that are specified as input arguments.  While multiple data source folders can be specified, <b>no two data sources can have the same name</b>.
 </p>
 <h4>Data Source Folders</h4>
 <p>
     In each main data source folder, there should be sub-directories for each individual data source, with further sub-directories for a specific reference (i.e. <i>hg19</i> or <i>hg38</i>).
     In the reference-specific data source directory, there is a configuration file detailing information about the data source and how to match it to a variant.  This configuration file is required.
 </p>
 <p>
     An example of a data source directory is the following:

     <pre>
         dataSourcesFolder/
              Data_Source_1/
                  hg19
                      data_source_1.config
                      data_source_1.data.file.one
                      data_source_1.data.file.two
                      data_source_1.data.file.three
                      ...
                   hg38
                      data_source_1.config
                      data_source_1.data.file.one
                      data_source_1.data.file.two
                      data_source_1.data.file.three
                      ...
              Data_Source_2/
                  hg19
                      data_source_2.config
                      data_source_2.data.file.one
                      data_source_2.data.file.two
                      data_source_2.data.file.three
                      ...
                   hg38
                      data_source_2.config
                      data_source_2.data.file.one
                      data_source_2.data.file.two
                      data_source_2.data.file.three
                      ...
               ...
     </pre>
 </p>
 <h4>Pre-packaged Data Sources</h4>
 <p>
     The GATK includes two sets of pre-packaged data sources, allowing for Funcotator use without (much) additional configuration.
     These data source packages correspond to the <strong>germline</strong> and <strong>somatic</strong> use cases.
     Broadly speaking, if you have a <strong>germline VCF</strong>, the <strong>germline data sources</strong> are what you want to use to start with.
     Conversely, if you have a <strong>somatic VCF</strong>, the <strong>somatic data sources</strong> are what you want to use to start with.

     <br /><b>Versioned gzip archives of data source files are provided here:</b>
     <ul>
         <li><a href="ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/funcotator/">ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/funcotator/</a></li>
         <li><a href="https://console.cloud.google.com/storage/browser/broad-public-datasets/funcotator">gs://broad-public-datasets/funcotator/</a></li>
     </ul>
 </p>

 <h5>gnomAD</h5>
 <p>
     The pre-packaged data sources include gnomAD, a large database of known variants.  gnomAD is split into two parts - one based on exome data, one based on whole genome data.
     Due to the size of gnomAD, it cannot be included in the data sources package directly.  Instead, the configuration data are present and point to a Google bucket in which
     the gnomAD data reside.  This will cause <i>Funcotator</i> to actively connect to that bucket when it is run.  <br/>
     For this reason, <strong>gnomAD is disabled by default</strong>.<br />
     To enable gnomAD, simply change directories to your data sources directory and untar the gnomAD tar.gz files:
     <pre>
         cd DATA_SOURCES_DIR
         tar -zxf gnomAD_exome.tar.gz
         tar -zxf gnomAD_genome.tar.gz
     </pre>
 </p>
 <p>
     Because <i>Funcotator</i> will query the Internet when gnomAD is enabled, performance will be impacted by the machine's Internet connection speed.
     If this degradation is significant, you can localize gnomAD to the machine running <i>Funcotator</i> to improve performance (however due to the size of gnomAD this may be impractical).
 </p>

 <h4>Data Source Downloader Tool</h4>
 <p>
     To improve ease-of-use of <b><i>Funcotator</i></b>, there is a tool to download the pre-packaged data sources to the user's machine.
     This tool is the <strong><i>FuncotatorDataSourceDownloader</i></strong> and can be run to retrieve the pre-packaged data sources from the google bucket and localize them to the machine on which it is run.
     <br />Briefly:
     <ul>
         <li>For <strong>somatic</strong> data sources:<br /><pre> ./gatk FuncotatorDataSourceDownloader --somatic --validate-integrity --extract-after-download</pre></li>
         <li>For <strong>germline</strong> data sources:<br /><pre> ./gatk FuncotatorDataSourceDownloader --germline --validate-integrity --extract-after-download</pre></li>
     </ul>
 </p>
 <h4>Disabling Data Sources</h4>
 <p>
     A data source can be disabled by removing the folder containing the configuration file for that source.  This can be done on a per-reference basis.  If the entire data source should be disabled, the entire top-level data source folder can be removed.
 </p>
 <p>
     If it is possible that the data source will be re-enabled in the future, then we recommend zipping the data source folder and removing the folder itself, leaving only the zip file in its place.  When the time comes to enable data source again, simply unzip the file and the data source will be ready to go the next time <i>Funcotator</i> is run.
 </p>
 <h4>User-Defined Data Sources</h4>
 <p>
     Users can define their own data sources by creating a new correctly-formatted data source sub-directory in the main data sources folder.  In this sub-directory, the user must create an additional folder for the reference for which the data source is valid.  If the data source is valid for multiple references, then multiple reference folders should be created.
     Inside each reference folder, the user should place the file(s) containing the data for the data source.  Additionally the user <b>must</b> create a configuration file containing metadata about the data source.
 </p>
 <p>
     <i>Funcotator</i> allows for data sources with source files that live on the cloud, enabling users to annotate with data sources that are not physically present on the machines running <i>Funcotator</i>.<br />
     To create a data source based on the cloud, create a configuration file for that data source and put the cloud URL in as the src_file property (see <i>Configuration File Format</i> for details).<br />
     E.g.:
     <pre>
         ...
         src_file = gs://broad-references/hg19/v0/1000G_phase1.snps.high_confidence.b37.vcf.gz
         ...
     </pre>
 </p>
 <p>
     There are several formats allowed for data sources, however the two most useful are arbitrarily separated value (XSV) files, such as comma-separated value (CSV), tab-separated value (TSV).  These files contain a table of data that can be matched to a variant by <i>gene name</i>, <i>transcript ID</i>, or <i>genome position</i>.
     In the case of <i>gene name</i> and <i>transcript ID</i>, one column must contain the <i>gene name</i> or <i>transcript ID</i> for each row's data.
     <ul>
         <li>For <i>gene name</i>, when a variant is annotated with a gene name that <i>exactly matches</i> an entry in the gene name column for a row, that row's other fields will be added as annotations to the variant.</li>
         <li>For <i>transcript ID</i>, when a variant is annotated with a transcript ID that <i>exactly matches</i> an entry in the transcript ID column for a row, that row's other fields will be added as annotations to the variant.</li>
         <li>For <i>genome position</i>, one column must contain the contig ID, another column must contain the start position (1-based, inclusive), and a column must contain the stop position (1-based, inclusive).  The start and stop columns may be the same column.  When a variant is annotated with a genome position that <i>overlaps</i> an entry in the three genome position columns for a row, that row's other fields will be added as annotations to the variant.</li>
     </ul>
 </p>
 <h4>Configuration File Format</h4>
 <p>
     The configuration file is a standard Java properties-style configuration file with key-value pairs.  This file name <b>must end in .config</b>.
 </p>
 <p>
     The following is an example of a genome position XSV configuration file (for the ORegAnno data source):
     <pre>
         name = Oreganno
         version = 20160119
         src_file = oreganno.tsv
         origin_location = http://www.oreganno.org/dump/ORegAnno_Combined_2016.01.19.tsv
         preprocessing_script = getOreganno.py

         # Supported types:
         # simpleXSV    -- Arbitrary separated value table (e.g. CSV), keyed off Gene Name OR Transcript ID
         # locatableXSV -- Arbitrary separated value table (e.g. CSV), keyed off a genome location
         # gencode      -- Custom datasource class for GENCODE
         # cosmic       -- Custom datasource class for COSMIC
         # vcf          -- Custom datasource class for Variant Call Format (VCF) files
         type = locatableXSV

         # Required field for GENCODE files.
         # Path to the FASTA file from which to load the sequences for GENCODE transcripts:
         gencode_fasta_path =

         # Required field for GENCODE files.
         # NCBI build version (either hg19 or hg38):
         ncbi_build_version =

         # Required field for simpleXSV files.
         # Valid values:
         #     GENE_NAME
         #     TRANSCRIPT_ID
         xsv_key =

         # Required field for simpleXSV files.
         # The 0-based index of the column containing the key on which to match
         xsv_key_column =

         # Required field for simpleXSV AND locatableXSV files.
         # The delimiter by which to split the XSV file into columns.
         xsv_delimiter = \t

         # Required field for simpleXSV files.
         # Whether to permissively match the number of columns in the header and data rows
         # Valid values:
         #     true
         #     false
         xsv_permissive_cols = true

         # Required field for locatableXSV files.
         # The 0-based index of the column containing the contig for each row
         contig_column = 1

         # Required field for locatableXSV files.
         # The 0-based index of the column containing the start position for each row
         start_column = 2

         # Required field for locatableXSV files.
         # The 0-based index of the column containing the end position for each row
         end_column = 3
     </pre>
 </p>

 <h3>Required Inputs</h3>
 <ul>
     <li>A reference genome sequence.</li>
     <li>The version of the reference genome sequence being used (e.g. <i>hg19</i>, <i>hg38</i>, etc.).</li>
     <li>A VCF of variant calls to annotate.</li>
     <li>The path to a folder of data sources formatted for use by Funcotator.</li>
     <li>The desired output format for the annotated variants file (either <i>MAF</i> or <i>VCF</i>)</li>
 </ul>

 <h3>Output</h3>
 <p>
     The basic output of <i>Funcotator</i> is:
 <ul>
     <li>A VCF or MAF file containing all variants from the input file with added annotations corresponding to annotations from each data source that matched a given variant according to that data source's matching criteria.</li>
 </ul>
 </p>

 <h4>Annotations for Pre-Packaged Data Sources</h4>
 <p>
     The pre-packaged data sources will create a set of baseline, or default annotations for an input data set.
     Most of these data sources copy and paste values from their source files into the output of <i>Funcotator</i> to create annotations.  In this sense they are trivial data sources.
 </p>
 <h5>Gencode</h5>
 <p>
     <i>Funcotator</i> performs some processing on the input data to create the Gencode annotations.  Gencode is currently required, so <i>Funcotator</i> will create these annotations for all input variants.
     The order and a specification of the Gencode annotations that <i>Funcotator</i> creates is as follows:
     <ol>
         <li>
             <b><i>hugoSymbol</i></b><br />
             <i>Type: </i>String<br />
             The name of the gene in which the annotated variant allele occurs.  If the variant allele occurs outside of any known gene boundaries, then this field is set to "Unknown".
         </li>
         <li>
             <b><i>ncbiBuild</i></b><br />
             <i>Type: </i>String<br />
             The reference which was used to create this Gencode annotation.  Current valid values are: "<i>hg19</i>" or "<i>hg38</i>".
         </li>
         <li>
             <b><i>chromosome</i></b><br />
             <i>Type: </i>String<br />
             The contig in which the variant occurs.  Will always correspond to the contig in the variant position.
         </li>
         <li>
             <b><i>start</i></b><br />
             <i>Type: </i>Integer<br />
             The start position in genomic coordinates of the variant allele being annotated (1-based, inclusive).  Will always correspond to the start in the variant position.
         </li>
         <li>
             <b><i>end</i></b><br />
             <i>Type: </i>Integer<br />
             The end position in genomic coordinates of the variant allele being annotated (1-based, inclusive).  Will always correspond to the position last base in the variant allele.
         </li>
         <li>
             <b><i>variantClassification</i></b><br />
             <i>Type: </i>String<br />
             The classification of the variant being annotated.  Will always be one of the following:
             <ul>

                  <li><p><i>COULD_NOT_DETERMINE</i><br />
                  Variant classification could not be determined.</p></li>

                  <li><p><i>INTRON</i><br />
                  Variant lies between exons within the bounds of the chosen transcript.
                  Only valid for Introns.</p></li>

                  <li><p><i>FIVE_PRIME_UTR</i><br />
                  Variant is on the 5'UTR for the chosen transcript.
                  Only valid for UTRs.</p></li>

                  <li><p><i>THREE_PRIME_UTR</i><br />
                  Variant is on the 3'UTR for the chosen transcript
                  Only valid for UTRs.</p></li>

                  <li><p><i>IGR</i><br />
                  Intergenic region. Does not overlap any transcript.
                  Only valid for IGRs.</p></li>

                  <li><p><i>FIVE_PRIME_FLANK</i><br />
                  The variant is upstream of the chosen transcript
                  Only valid for IGRs.</p></li>

                  <li><p><i>THREE_PRIME_FLANK</i><br />
                  The variant is downstream of the chosen transcript
                  Only valid for IGRs.</p></li>

                  <li><p><i>MISSENSE</i><br />
                  The point mutation alters the protein structure by one amino acid.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>NONSENSE</i><br />
                  A premature stop codon is created by the variant.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>NONSTOP</i><br />
                  Variant removes stop codon.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>SILENT</i><br />
                  Variant is in coding region of the chosen transcript, but protein structure is identical.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>SPLICE_SITE</i><br />
                  The variant is within a configurable number of bases  of a splice site. See the secondary classification to determine if it lies on the exon or intron side.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>IN_FRAME_DEL</i><br />
                  Deletion that keeps the sequence in frame.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>IN_FRAME_INS</i><br />
                  Insertion that keeps the sequence in frame.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>FRAME_SHIFT_INS</i><br />
                  Insertion that moves the coding sequence out of frame.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>FRAME_SHIFT_DEL</i><br />
                  Deletion that moves the sequence out of frame.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>START_CODON_SNP</i><br />
                  Point mutation that overlaps the start codon.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>START_CODON_INS</i><br />
                  Insertion that overlaps the start codon.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>START_CODON_DEL</i><br />
                  Deletion that overlaps the start codon.
                  Can occur in Coding regions or Introns.</p></li>

                  <li><p><i>DE_NOVO_START_IN_FRAME</i><br />
                  New start codon is created by the given variant using the chosen transcript.
                  However, it is in frame relative to the coded protein, meaning that if the coding sequence were extended
                  then the new start codon would be in frame with the
                  existing start and stop codons.

                  This can only occur in a 5' UTR.</p></li>

                  <li><p><i>DE_NOVO_START_OUT_FRAME</i><br />
                  New start codon is created by the given variant using the chosen transcript.
                  However, it is out of frame relative to the coded protein, meaning that if the coding sequence were extended
                  then the new start codon would NOT be in frame with
                  the existing start and stop codons.

                  This can only occur in a 5' UTR.</p></li>

                  <li><p><i>RNA</i><br />
                  Variant lies on one of the RNA transcripts.
                  (special catch-all case)</p></li>

                  <li><p><i>LINCRNA</i><br />
                  Variant lies on one of the lincRNAs.
                  (special catch-all case)</p></li>
             </ul>
         </li>
         <li>
             <b><i>secondaryVariantClassification</i></b><br />
             <i>Type: </i>String<br />
             Additional variant classification information for variant alleles that have a variantClassification of SPLICE_SITE.
             For a variant allele with the variantClassification of SPLICE_SITE, this will indicate the specific classification of the variant.
             For all variants that do not have the variantClassification of SPLICE_SITE, this will be the empty string.
         </li>
         <li>
             <b><i>variantType</i></b><br />
             <i>Type: </i>String<br />
             Basic information about the variant allele being annotated.  Can be one of:
             <ul>
                 <li><i>INS</i> - The variant allele is some kind of insertion.</li>
                 <li><i>DEL</i> - The variant allele is some kind of deletion.</li>
                 <li><i>SNP</i> - The variant allele is a single nucleotide polymorphism.</li>
                 <li><i>DNP</i> - The variant allele is a di-nucleotide polymorphism.</li>
                 <li><i>TNP</i> - The variant allele is a tri-nucleotide polymorphism.</li>
                 <li><i>ONP</i> - The variant allele is an oligo-nucleotide polymorphism (Synonymous with MNP).</li>
                 <li><i>MNP</i> - The variant allele is a multi-nucleotide polymorphism (Synonymous with ONP).</li>
                 <li><i>NA</i> - The variant allele type cannot be determined.</li>
             </ul>
         </li>
         <li>
             <b><i>refAllele</i></b><br />
             <i>Type: </i>String<br />
             The reference allele for the position at which this this variant allele occurs.<br />
             For insertions, this will be set to <i>"-"</i>.
         </li>
         <li>
             <b><i>tumorSeqAllele1</i></b><br />
             <i>Type: </i>String<br />
             Always the same as the reference allele.  This field is a hold-over required for MAF annotations.<br />
             For insertions, this will be set to <i>"-"</i>.
         </li>
         <li>
             <b><i>tumorSeqAllele2</i></b><br />
             <i>Type: </i>String<br />
             The variant allele being annotated.  This field only includes the bases that are different from the reference.<br />
             For the input VCF records, this field may slightly differ from the alternate allele reported in the base data for the VariantContext.<br />
             For deletions, this will be set to <i>"-"</i>.
         </li>
         <li>
             <b><i>genomeChange</i></b><br />
             <i>Type: </i>String<br />
             A String summarizing the change resulting from this variant allele within the context of the whole genome sequence.<br />
             Generally the format of this field is:<br />
             <pre>
             g.[CONTIG]:[POSITION][BASES CHANGED]
             </pre>

             The format of this field slightly varies based on VariantType:
             <ul>
                 <li>
                     <i><u>Insertion</u></i><br />
                     <pre>g.[CONTIG]:[POSITION OF BASE PRIOR TO INSERTION];_[POSITION OF BASE AFTER INSERTION]ins[BASES INSERTED]</pre><br />
                     E.g.:<br />
                     <pre>g.chr19:2018023_2018024insAATCG</pre><br />
                     This indicates that the bases AATCG were inserted between bases 2018023 and 2018024 on chromosome 19.
                 </li>
                 <li>
                     <i><u>Deletion</u></i><br />
                     <pre>g.[CONTIG]:[POSITION OF BASE DELETED]del[BASE DELETED]</pre><br />
                     E.g.:<br />
                     <pre>g.chr19:2018023delT</pre><br />
                     This indicates that the base T was deleted at position 2018023 on chromosome 19.
                     <i>OR</i>
                     <pre>g.[CONTIG]:[POSITION OF FIRST BASE DELETED]_[POSITION OF LAST BASE DELETED]del[BASES DELETED]</pre><br />
                     E.g.:<br />
                     <pre>g.chr19:2018023_2018025delTTG</pre><br />
                     This indicates that the bases TTG were deleted starting at position 2018023 and ending at position 2018025 on chromosome 19.
                 </li>
                 <li>
                     <i><u>SNP</u></i><br />
                     <pre>g.[CONTIG]:[POSITION OF BASE ALTERED][REFERENCE BASE]>[ALTERNATE BASE]</pre><br />
                     E.g.:<br />
                     <pre>g.chr19:2018023T>G</pre><br />
                     This indicates that the base T was changed to G at position 2018023 on chromosome 19.
                 </li>
                 <li>
                     <i><u>MNP (including DNPs, TNPs)</u></i><br />
                     <pre>g.[CONTIG]:[POSITION OF FIRST BASE ALTERED]_[POSITION OF LAST BASE ALTERED][REFERENCE BASES>&gt;[ALTERNATE BASES]</pre><br />
                     E.g.:<br />
                     <pre>g.chr19:2018023_2018025TTG>GAT</pre><br />
                     This indicates that the bases TTG were changed to GAT from position 2018023 to position 2018025 on chromosome 19.
                 </li>
             </ul>
         </li>
         <li>
             <b><i>annotationTranscript</i></b><br />
             <i>Type: </i>String<br />
             The ID of the transcript chosen for the detailed Gencode annotation reporting.<br />E.g.: <i>ENST00000435064.1</i><br />
             If the variant allele does not occur within the bounds of any transcript (e.g. is of type <i>IGR</i>), then this field is empty.
         </li>
         <li>
             <b><i>transcriptStrand</i></b><br />
             <i>Type: </i>String<br />
             The strand direction associated with the transcript on which this variant allele occurs.<br />
             Either "<i>+</i>" or "<i>-</i>".
         </li>
         <li>
             <b><i>transcriptExon</i></b><br />
             <i>Type: </i>Integer or Empty<br />
             The exon number on the transcript in which this variant allele occurs (1-based).  Corresponds directly to the Gencode exon number.<br />
             If the variant does not occur in the expressed transcript of the corresponding gene (e.g. is of type <i>INTRON</i> or <i>IGR</i>), then this field is empty.
         </li>
         <li>
             <b><i>transcriptPos</i></b><br />
             <i>Type: </i>Integer or Empty<br />
             Position in the chosen transcript of the variant allele.  All positions listed are 1-based and inclusive (meaning that the first base in the transcript starts at and ends at position 1).<br />
             For variant alleles that occur at a single base, the format is simply the position at which that variant occurs in the transcript (e.g. <i>1294</i>)<br />
             For variant alleles spanning multiple bases, the format is:<br />
             <pre>
             [START]_[END]
             </pre>
             E.g.: <pre>1236_1237</pre><br />
             If the variant does not occur in the expressed transcript of the corresponding gene (e.g. is of type <i>INTRON</i> or <i>IGR</i>), then this field is empty.
         </li>
         <li>
             <b><i>cDnaChange</i></b><br />
             <i>Type: </i>String<br />
             A String that summarizes the change resulting from this variant allele in the coding sequence for the transcript in which it occurs.<br />
             Positions in this field are <i>relative to the start of the transcript (1-based, inclusive)</i> unless otherwise noted.<br />
             Generally the format of this field is:<br />
             <pre>
             c.[POSITION][BASES CHANGED]
             </pre>

             The format of this field slightly varies based on VariantType, the number of affected bases, and whether the variant allele is a <i>SPLICE_SITE</i>:
             <ul>
                 <li>
                     <i><u>Insertions</u></i><br />
                     <pre>c.[POSITION OF BASE PRIOR TO INSERTION]_[POSITION OF BASE AFTER INSERTION]ins[BASES INSERTED]</pre><br />
                     E.g.:<br />
                     <pre>c.2018_2019insAA</pre><br />
                     This indicates that the bases AA were inserted between bases 2018 and 2019 in the transcript associated with this variant allele.
                 </li>
                 <li>
                     <i><u>Deletions of One Base</u></i><br />
                     <pre>c.[POSITION OF BASE DELETED]del[BASE DELETED]</pre><br />
                     E.g.:<br />
                     <pre>c2018delT</pre><br />
                     This indicates that the base T was deleted at position 2018 in the transcript associated with this variant allele.
                 </li>
                 <li>
                     <i><u>Deletions of Multiple Bases</u></i><br />
                     <pre>c.[POSITION OF FIRST BASE DELETED]_[POSITION OF LAST BASE DELETED]del[BASES DELETED]</pre><br />
                     E.g.:<br />
                     <pre>c2018_2022delTTCAG</pre><br />
                     This indicates that the bases TTCAG were deleted from position 2018 to position 2022 in the transcript associated with this variant allele.
                 </li>
                 <li>
                     <i><u>SNPs</u></i><br />
                     <pre>c.[POSITION OF BASE CHANGED]&gt;[NEW BASE]</pre><br />
                     E.g.:<br />
                     <pre>c.1507T>G</pre><br />
                     This indicates that the base T was changed to G at position 1507 in the transcript associated with this variant allele.
                 </li>
                 <li>
                     <i><u>MNPs (including DNPs, TNPs)</u></i><br />
                     <pre>c.[POSITION OF FIRST BASE CHANGED]_[POSITION OF LAST BASE CHANGED]&gt;[NEW BASES]</pre><br />
                     E.g.:<br />
                     <pre>c.12899_12900AG>TA</pre><br />
                     This indicates that the bases AG were changed to TA from position 12899 to position 12900 in the transcript associated with this variant allele.
                 </li>
                 <li>
                     <i><u>INTRONIC Variants</u></i><br />
                     For variants occuring in INTRONs, the cDNA string position indicates the offset from the exon boundary for the start of the variant.  The whole string takes the form:<br />
                     <pre>c.e[EXON NUMBER][+|-][BASES FROM EXON][REF ALLELE]>[ALT ALLELE]</pre><br />
                     E.g.:<br />
                     <pre>c.e81-4TAA>A</pre><br />
                     This indicates that the bases TAA were changed to A starting four bases before exon 81 in the transcript associated with this variant allele.
                 </li>
             </ul>
             If the variant does not occur in the expressed transcript of the corresponding gene (e.g. is of type <i>IGR</i>), then this field is empty.
         </li>
         <li>
             <b><i>codonChange</i></b><br />
             <i>Type: </i>String<br />
             A String that representing the <i>codon-aligned change</i> resulting from this variant allele in the coding sequence for the transcript in which it occurs.<br />
             Positions in this field are <i>relative to the start of the transcript (1-based, inclusive) and aligned to the coding sequence</i>unless otherwise noted.<br />
             Unlike the cDnaChange, the bases reported in the codonChange string will always have a length evenly divisble by 3 (except for frameshifts) and represent what the codons would be if the variant alternate allele were expressed in the reference sequence.<br />
             Capitalized bases represent the bases changed by the variant alternate allele.  Lower-case bases represent reference bases.<br />
             Generally the format of this field is:<br />
             <pre>
             c.[POSITION][BASES CHANGED]
             </pre>
             The format of this field slightly varies based on VariantType, the number of affected bases, and whether the variant allele occurs in an Intron:
             <ul>
                 <li>
                     <i><u>In-Frame Insertions</u></i><br />
                     <pre>c.([POSITION OF FIRST BASE IN FIRST CODON IN THE REFERENCE AFFECTED BY THIS VARIANT]-[POSITION OF LAST BASE IN LAST CODON IN THE REFERENCE AFFECTED BY THIS VARIANT][REFERENCE CODONS]&gt;[EXPRESSED CODONS]</pre><br />
                     E.g.:<br />
                     <pre>c.(19-21)ctt>ctCGTt</pre><br />
                     This indicates that the bases <i>CGT</i> were inserted before the 6th codon (starting at base 19, ending at base 21) in the transcript associated with this variant allele, and the resulting expressed codons would be <i>ctCGTt</i>.
                 </li>
                 <li>
                     <i><u>In-Frame Deletions of Complete Codons (codon-aligned deletions)</u></i><br />
                     <pre>c.([POSITION OF FIRST BASE IN FIRST CODON DELETED]-[POSITION OF LAST BASE IN LAST CODON DELETED][REFERENCE CODONS]del</pre><br />
                     E.g.:<br />
                     <pre>c.(997-999)gcadel</pre><br />
                     This indicates that the 332nd codon (starting at base 997, ending at base 999) was deleted in the transcript associated with this variant allele, and the deleted codon bases are <i>gca</i>.
                 </li>
                 <li>
                     <i><u>In-Frame Deletions Spanning Multiple Codons</u></i><br />
                     <pre>c.([POSITION OF FIRST BASE IN FIRST CODON DELETED]-[POSITION OF LAST BASE IN LAST CODON DELETED][REFERENCE CODONS]&gt;[EXPRESSED CODONS]</pre><br />
                     E.g.:<br />
                     <pre>c.(997-1002)gcactc>gtc</pre><br />
                     This indicates that bases in the 332nd codon (starting at base 997) and 333rd codon (ending at base 1002) were deleted in the transcript associated with this variant allele, and the resulting expressed codon would be <i>gtc</i>.
                 </li>
                 <li>
                     <i><u>Frame Shift Insertions and Deletions</u></i><br />
                     <pre>c.([POSITION OF FIRST BASE IN LAST CORRECTLY EXPRESSED/REFERENCE CODON]-[POSITION OF LAST BASE IN LAST CORRECTLY EXPRESSED/REFERENCE CODON][REFERENCE CODONS]&gt;[EXPRESSED CODONS]</pre><br />
                     E.g.:<br />
                     <pre>c.(997-999)gcafs</pre><br />
                     This indicates that bases just <u><i>AFTER</i></u> the 332nd codon (starting at base 997, ending at base 999) were inserted or deleted in the transcript associated with this variant allele resulting in a frame shift, and that the last correctly transcribed codon would be codon 332 (starting at base 997, ending at base 999), '<i>gca</i>'.
                 </li>
                 <i><u>SNPs / MNPs</u></i><br />
                     <pre>c.([POSITION OF FIRST BASE IN FIRST CODON IN THE REFERENCE AFFECTED BY THIS VARIANT]-[POSITION OF LAST BASE IN LAST CODON IN THE REFERENCE AFFECTED BY THIS VARIANT][REFERENCE CODONS]&gt;[EXPRESSED CODONS]</pre><br />
                     E.g. 1:<br />
                     <pre>c.(39871-39873)cCC>cTT</pre><br />
                     This indicates that the bases <i>CC</i> were changed to <i>TT</i> in the 13290th codon (starting at base 39871, ending at base 39873) in the transcript associated with this variant allele, and the resulting expressed codon would be <i>cTT</i>.<br />
                     E.g. 2:<br />
                     <pre>c.(4-9)ctAAgc>ctGCgc</pre><br />
                     This indicates that the bases <i>AA</i> starting in the 2nd codon (starting at base 4) and ending in the 3rd codon (ending at base 9) were changed to <i>GC</i> in the transcript associated with this variant allele, and the resulting expressed codons would be <i>ctGCgc</i>.
                 </li>
             </ul>
         </li>
         <li>
             <b><i>proteinChange</i></b><br />
             <i>Type: </i>String<br />
             A short string representing the predicted amino acid sequence change in the product of the gene transcript in which this variant alternate allele occurs.<br />
             Positions in this field are <i>relative to the start of the amino acid sequence (1-based, inclusive) resulting from decoding the codons in the transcript in which this variant alternate allele occurs</i>unless otherwise noted.<br />
             Amino acid abbreviations are the standard letters as can be found in <a href="https://en.wikipedia.org/wiki/DNA_codon_table"></a>this table</a> with the exception of the stop codon, which is represented by '<i><b>*</b></i>'.<br />
             It is important to note that the positions and amino acids reported in this string may not directly align to the codons in which the variant alternate allele occurs.  This is most often due to the variant occuring in a set of tandem repeats which would cause the amino acid change to be "pushed" to the end of the tandem repeats.<br />
             For protein change strings in the Mitochondrial contig, <a href="https://en.wikipedia.org/wiki/Vertebrate_mitochondrial_code">the mitochondrial genetic code</a> is used (rather than the standard code).
             The format of this field takes two forms:<br />
             <ul>
                 <li>
                     <i>Protein Changes with One Amino Acid Changed</i><br />
                     <pre>p.[REFERENCE AMINO ACID][POSITION][PREDICTED EXPRESSED AMINO ACID]</pre><br />
                     E.g. 1:<br />
                     <pre>p.V5T</pre><br />
                     The amino acid at protein position 5 was V (Valine) in the reference and would become T (Threonine) with the variant alternate allele expressed.<br />
                     E.g. 2:<br />
                     <pre>p.R2R</pre><br />
                     The amino acid at protein position 2 was R (Argenine) in the reference and would become R (Argenine) with the variant alternate allele expressed (no change in amino acid sequence / silent variant classification).
                 </li>
                 <li>
                     <i>Protein Changes with Multiple Amino Acids Changed</i><br />
                     <pre>p.[FIRST AFFECTED AMINO ACID POSITION]_[LAST AFFECTED AMINO ACID POSITION][REFERENCE AMINO ACIDS]&gt;[PREDICTED EXPRESSED AMINO ACIDS]</pre><br />
                     E.g.:<br />
                     <pre>p.100_101Q*>FL</pre><br />
                     The amino acid sequence starting at protein position 100 and ending at protein position 101 was Q* (Glutamine,STOP) in the reference and would become FL (Phenylalanine,Leucine) with the variant alternate allele expressed.
                 </li>
             </ul>
             If the variant alternate allele does not occur in a coding region, this field will be empty.
         </li>
         <li>
             <b><i>gcContent</i></b><br />
             <i>Type: </i>Double<br />
             Represents the fraction of Guanine and Cytosine bases in a window of a given size around a variant.  This window size does not include any bases in the variant alternate allele itself.  By default the window size is 200 bases.
         </li>
         <li>
             <b><i>referenceContext</i></b><br />
             <i>Type: </i>String<br />
             The <u>strand-correct</u> reference coding sequence in a given window around the reference allele.  By default the window size is <i>10 bases</i>.<br />
             E.g. For the reference context around a variant with the reference allele '<i>C</i>' on the '+' strand:

             <pre>
                      [REF ALLELE]
                           |
                           v
                 GAACCCACGTCGGTGAGGGCC
                 |________| |________|
                     v           v
                  10 bases      10 bases
                (window size)  (window size)
             </pre>
             Strand-correct specifically means that if the strand of this transcript is determined to be '<i>-</i>' then the sequence is reverse complemented.  <br />
             E.g. For the reference context around a variant with the reference allele '<i>C</i>' on the '-' strand:
             <pre>
                      [REF ALLELE]
                           |
                           v
                 CACGAAAGTCTTGCGGATCT
                 |________| |________|
                     v           v
                  10 bases      10 bases
                (window size)  (window size)
             </pre>
         </li>
         <li>
             <b><i>otherTranscripts</i></b><br />
             <i>Type: </i>String<br />
             A summary of the other transcripts in which this variant occurs, which were not chosen for detailed reporting due to the transcript selection scheme.<br />
             Each other transcript is represented by a condensed string that indicates how that transcript would be affected by this variant alternate allele.<br />
             Each other transcript takes the form:<br />
             <pre>[HUGO SYMBOL]_[TRANSCRIPT ID]_[VARIANT CLASSIFICATION]_[PROTEIN CHANGE STRING]</pre><br />
             E.g.:
             <pre>SDF4_ENST00000263741.7_MISSENSE_p.R243Q</pre>
             If another transcript were to be an IGR, the other transcript field would be populated with '<i>IGR_ANNOTATON</i>'<br />
             If the other transcript does not have a protein change string, then that part is not rendered.<br />
             In the event that there are multiple other transcripts, these transcripts are separated by '<i>/</i>'.<br />
             E.g.:
             <pre>SDF4_ENST00000263741.7_MISSENSE_p.R243Q/TNFRSF4_ENST00000379236.3_FIVE_PRIME_FLANK</pre>
             If this variant alternate allele occurs in only one transcript, this field will be empty.
         </li>
     </ol>
 </p>
 <p>
     Other annotations will follow the Gencode annotations and will be based on the data sources included in the data sources directory.
 </p>
 <h3>Usage example</h3>
 <pre>
   ./gatk Funcotator \
   -R reference.fasta \
   -V input.vcf \
   -O outputFile \
   --output-file-format MAF \
   --data-sources-path dataSourcesFolder/ \
   --ref-version hg19
 </pre>

 <h3>Notes</h3>
 <ul>
     <li>This tool is the spiritual successor to <a href="https://github.com/broadinstitute/oncotator">Oncotator</a>, with better support for germline data, numerous fixes for correctness, and many other features.</li>
     <li>REMEMBER: <strong>Funcotator is NOT Oncotator.</strong></li>
 </ul>

 <h3>Known Issues</h3>
 <p>A complete list of known open issues can be found on <a href="https://github.com/broadinstitute/gatk/issues?q=is%3Aopen+is%3Aissue+label%3AFuncotator">the GATK github entry for funcotator here.</a></p>

				<hr>
				<h2>Additional Information</h2>
				<p></p>
				<h3>Read filters</h3>
					<p>This Read Filter is automatically applied to the data by the Engine before processing by Funcotator.</p>
				<ul>
						<li><a href="org_broadinstitute_hellbender_engine_filters_WellformedReadFilter.html">WellformedReadFilter</a></li>
				</ul>

				<h3>Funcotator specific arguments</h3>
				<p>This table summarizes the command-line arguments that are specific to this tool. For more details on each argument, see the list further down below the table or click on an argument name to jump directly to that entry in the list.</p>
				<table class="table table-striped table-bordered table-condensed cozy">
					<thead>
					<tr>
						<th>Argument name(s)</th>
						<th>Default value</th>
						<th>Summary</th>
					</tr>
					</thead>
					<tbody>
			<tr>
				<th colspan="4" id="row-divider">Required Arguments</th>
			</tr>
				<tr>
					<td><a href="#--data-sources-path">--data-sources-path</a><br />
					</td>
					<!--<td>List[String]</td> -->
						<td></td>
					<td>The path to a data source folder for Funcotator.  May be specified more than once to handle multiple data source folders.</td>
				</tr>
				<tr>
					<td><a href="#--output">--output</a><br />
								&nbsp;<em>-O</em>
					</td>
					<!--<td>File</td> -->
						<td></td>
					<td>Output file to which annotated variants should be written.</td>
				</tr>
				<tr>
					<td><a href="#--output-file-format">--output-file-format</a><br />
					</td>
					<!--<td>OutputFormatType</td> -->
						<td></td>
					<td>The output file format.  Either VCF, MAF, or SEG.  Please note that MAF output for germline use case VCFs is unsupported.  SEG will generate two output files: a simple tsv and a gene list.</td>
				</tr>
				<tr>
					<td><a href="#--ref-version">--ref-version</a><br />
					</td>
					<!--<td>String</td> -->
						<td></td>
					<td>The version of the Human Genome reference to use (e.g. hg19, hg38, etc.).  This will correspond to a sub-folder of each data source corresponding to that data source for the given reference.</td>
				</tr>
				<tr>
					<td><a href="#--reference">--reference</a><br />
								&nbsp;<em>-R</em>
					</td>
					<!--<td>GATKPath</td> -->
						<td></td>
					<td>Reference sequence file</td>
				</tr>
				<tr>
					<td><a href="#--variant">--variant</a><br />
								&nbsp;<em>-V</em>
					</td>
					<!--<td>GATKPath</td> -->
						<td></td>
					<td>A VCF file containing variants</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Tool Arguments</th>
			</tr>
				<tr>
					<td><a href="#--annotation-default">--annotation-default</a><br />
					</td>
					<!--<td>List[String]</td> -->
						<td></td>
					<td>Annotations to include in all annotated variants if the annotation is not specified in the data sources (in the format <ANNOTATION>:<VALUE>).  This will add the specified annotation to every annotated variant if it is not already present.</td>
				</tr>
				<tr>
					<td><a href="#--annotation-override">--annotation-override</a><br />
					</td>
					<!--<td>List[String]</td> -->
						<td></td>
					<td>Override values for annotations (in the format <ANNOTATION>:<VALUE>).  Replaces existing annotations of the given name with given values.</td>
				</tr>
				<tr>
					<td><a href="#--arguments_file">--arguments_file</a><br />
					</td>
					<!--<td>List[File]</td> -->
						<td></td>
					<td>read one or more arguments files and add them to the command line</td>
				</tr>
				<tr>
					<td><a href="#--cloud-index-prefetch-buffer">--cloud-index-prefetch-buffer</a><br />
								&nbsp;<em>-CIPB</em>
					</td>
					<!--<td>int</td> -->
						<td>-1</td>
					<td>Size of the cloud-only prefetch buffer (in MB; 0 to disable). Defaults to cloudPrefetchBuffer if unset.</td>
				</tr>
				<tr>
					<td><a href="#--cloud-prefetch-buffer">--cloud-prefetch-buffer</a><br />
								&nbsp;<em>-CPB</em>
					</td>
					<!--<td>int</td> -->
						<td>40</td>
					<td>Size of the cloud-only prefetch buffer (in MB; 0 to disable).</td>
				</tr>
				<tr>
					<td><a href="#--disable-bam-index-caching">--disable-bam-index-caching</a><br />
								&nbsp;<em>-DBIC</em>
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>If true, don't cache bam indexes, this will reduce memory requirements but may harm performance if many intervals are specified.  Caching is automatically disabled if there are no intervals specified.</td>
				</tr>
				<tr>
					<td><a href="#--disable-sequence-dictionary-validation">--disable-sequence-dictionary-validation</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>If specified, do not check the sequence dictionaries from our inputs for compatibility. Use at your own risk!</td>
				</tr>
				<tr>
					<td><a href="#--exclude-field">--exclude-field</a><br />
					</td>
					<!--<td>Set[String]</td> -->
						<td></td>
					<td>Fields that should not be rendered in the final output.  Only exact name matches will be excluded.</td>
				</tr>
				<tr>
					<td><a href="#--five-prime-flank-size">--five-prime-flank-size</a><br />
					</td>
					<!--<td>int</td> -->
						<td>5000</td>
					<td>Variants within this many bases of the 5' end of a transcript (and not overlapping any part of the transcript itself) will be annotated as being in the 5' flanking region of that transcript</td>
				</tr>
				<tr>
					<td><a href="#--gcs-max-retries">--gcs-max-retries</a><br />
								&nbsp;<em>-gcs-retries</em>
					</td>
					<!--<td>int</td> -->
						<td>20</td>
					<td>If the GCS bucket channel errors out, how many times it will attempt to re-initiate the connection</td>
				</tr>
				<tr>
					<td><a href="#--gcs-project-for-requester-pays">--gcs-project-for-requester-pays</a><br />
					</td>
					<!--<td>String</td> -->
						<td></td>
					<td>Project to bill when accessing "requester pays" buckets. If unset, these buckets cannot be accessed.  User must have storage.buckets.get permission on the bucket being accessed.</td>
				</tr>
				<tr>
					<td><a href="#--help">--help</a><br />
								&nbsp;<em>-h</em>
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>display the help message</td>
				</tr>
				<tr>
					<td><a href="#--interval-merging-rule">--interval-merging-rule</a><br />
								&nbsp;<em>-imr</em>
					</td>
					<!--<td>IntervalMergingRule</td> -->
						<td>ALL</td>
					<td>Interval merging rule for abutting intervals</td>
				</tr>
				<tr>
					<td><a href="#--intervals">--intervals</a><br />
								&nbsp;<em>-L</em>
					</td>
					<!--<td>List[String]</td> -->
						<td></td>
					<td>One or more genomic intervals over which to operate</td>
				</tr>
				<tr>
					<td><a href="#--lookahead-cache-bp">--lookahead-cache-bp</a><br />
					</td>
					<!--<td>int</td> -->
						<td>100000</td>
					<td>Number of base-pairs to cache when querying variants.  Can be overridden in individual data source configuration files.</td>
				</tr>
				<tr>
					<td><a href="#--reannotate-vcf">--reannotate-vcf</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>When input VCF has already been annotated, still annotate again.</td>
				</tr>
				<tr>
					<td><a href="#--remove-filtered-variants">--remove-filtered-variants</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>Ignore/drop variants that have been filtered in the input.  These variants will not appear in the output file.</td>
				</tr>
				<tr>
					<td><a href="#--sites-only-vcf-output">--sites-only-vcf-output</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>If true, don't emit genotype fields when writing vcf file output.</td>
				</tr>
				<tr>
					<td><a href="#--three-prime-flank-size">--three-prime-flank-size</a><br />
					</td>
					<!--<td>int</td> -->
						<td>0</td>
					<td>Variants within this many bases of the 3' end of a transcript (and not overlapping any part of the transcript itself) will be annotated as being in the 3' flanking region of that transcript</td>
				</tr>
				<tr>
					<td><a href="#--transcript-list">--transcript-list</a><br />
					</td>
					<!--<td>Set[String]</td> -->
						<td></td>
					<td>File to use as a list of transcripts (one transcript ID per line, version numbers are ignored) OR A set of transcript IDs to use for annotation to override selected transcript.</td>
				</tr>
				<tr>
					<td><a href="#--transcript-selection-mode">--transcript-selection-mode</a><br />
					</td>
					<!--<td>TranscriptSelectionMode</td> -->
						<td>CANONICAL</td>
					<td>Method of detailed transcript selection.  This will select the transcript for detailed annotation (CANONICAL, ALL, or BEST_EFFECT).</td>
				</tr>
				<tr>
					<td><a href="#--version">--version</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>display the version number for this tool</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Common Arguments</th>
			</tr>
				<tr>
					<td><a href="#--add-output-sam-program-record">--add-output-sam-program-record</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>true</td>
					<td>If true, adds a PG tag to created SAM/BAM/CRAM files.</td>
				</tr>
				<tr>
					<td><a href="#--add-output-vcf-command-line">--add-output-vcf-command-line</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>true</td>
					<td>If true, adds a command line header line to created VCF files.</td>
				</tr>
				<tr>
					<td><a href="#--create-output-bam-index">--create-output-bam-index</a><br />
								&nbsp;<em>-OBI</em>
					</td>
					<!--<td>boolean</td> -->
						<td>true</td>
					<td>If true, create a BAM/CRAM index when writing a coordinate-sorted BAM/CRAM file.</td>
				</tr>
				<tr>
					<td><a href="#--create-output-bam-md5">--create-output-bam-md5</a><br />
								&nbsp;<em>-OBM</em>
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>If true, create a MD5 digest for any BAM/SAM/CRAM file created</td>
				</tr>
				<tr>
					<td><a href="#--create-output-variant-index">--create-output-variant-index</a><br />
								&nbsp;<em>-OVI</em>
					</td>
					<!--<td>boolean</td> -->
						<td>true</td>
					<td>If true, create a VCF index when writing a coordinate-sorted VCF file.</td>
				</tr>
				<tr>
					<td><a href="#--create-output-variant-md5">--create-output-variant-md5</a><br />
								&nbsp;<em>-OVM</em>
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>If true, create a a MD5 digest any VCF file created.</td>
				</tr>
				<tr>
					<td><a href="#--disable-read-filter">--disable-read-filter</a><br />
								&nbsp;<em>-DF</em>
					</td>
					<!--<td>List[String]</td> -->
						<td></td>
					<td>Read filters to be disabled before analysis</td>
				</tr>
				<tr>
					<td><a href="#--disable-tool-default-read-filters">--disable-tool-default-read-filters</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>Disable all tool default read filters (WARNING: many tools will not function correctly without their default read filters on)</td>
				</tr>
				<tr>
					<td><a href="#--exclude-intervals">--exclude-intervals</a><br />
								&nbsp;<em>-XL</em>
					</td>
					<!--<td>List[String]</td> -->
						<td></td>
					<td>One or more genomic intervals to exclude from processing</td>
				</tr>
				<tr>
					<td><a href="#--gatk-config-file">--gatk-config-file</a><br />
					</td>
					<!--<td>String</td> -->
						<td></td>
					<td>A configuration file to use with the GATK.</td>
				</tr>
				<tr>
					<td><a href="#--input">--input</a><br />
								&nbsp;<em>-I</em>
					</td>
					<!--<td>List[GATKPath]</td> -->
						<td></td>
					<td>BAM/SAM/CRAM file containing reads</td>
				</tr>
				<tr>
					<td><a href="#--interval-exclusion-padding">--interval-exclusion-padding</a><br />
								&nbsp;<em>-ixp</em>
					</td>
					<!--<td>int</td> -->
						<td>0</td>
					<td>Amount of padding (in bp) to add to each interval you are excluding.</td>
				</tr>
				<tr>
					<td><a href="#--interval-padding">--interval-padding</a><br />
								&nbsp;<em>-ip</em>
					</td>
					<!--<td>int</td> -->
						<td>0</td>
					<td>Amount of padding (in bp) to add to each interval you are including.</td>
				</tr>
				<tr>
					<td><a href="#--interval-set-rule">--interval-set-rule</a><br />
								&nbsp;<em>-isr</em>
					</td>
					<!--<td>IntervalSetRule</td> -->
						<td>UNION</td>
					<td>Set merging approach to use for combining interval inputs</td>
				</tr>
				<tr>
					<td><a href="#--lenient">--lenient</a><br />
								&nbsp;<em>-LE</em>
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>Lenient processing of VCF files</td>
				</tr>
				<tr>
					<td><a href="#--max-variants-per-shard">--max-variants-per-shard</a><br />
					</td>
					<!--<td>int</td> -->
						<td>0</td>
					<td>If non-zero, partitions VCF output into shards, each containing up to the given number of records.</td>
				</tr>
				<tr>
					<td><a href="#--QUIET">--QUIET</a><br />
					</td>
					<!--<td>Boolean</td> -->
						<td>false</td>
					<td>Whether to suppress job-summary info on System.err.</td>
				</tr>
				<tr>
					<td><a href="#--read-filter">--read-filter</a><br />
								&nbsp;<em>-RF</em>
					</td>
					<!--<td>List[String]</td> -->
						<td></td>
					<td>Read filters to be applied before analysis</td>
				</tr>
				<tr>
					<td><a href="#--read-index">--read-index</a><br />
					</td>
					<!--<td>List[GATKPath]</td> -->
						<td></td>
					<td>Indices to use for the read inputs. If specified, an index must be provided for every read input and in the same order as the read inputs. If this argument is not specified, the path to the index for each input will be inferred automatically.</td>
				</tr>
				<tr>
					<td><a href="#--read-validation-stringency">--read-validation-stringency</a><br />
								&nbsp;<em>-VS</em>
					</td>
					<!--<td>ValidationStringency</td> -->
						<td>SILENT</td>
					<td>Validation stringency for all SAM/BAM/CRAM/SRA files read by this program.  The default stringency value SILENT can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) do not otherwise need to be decoded.</td>
				</tr>
				<tr>
					<td><a href="#--seconds-between-progress-updates">--seconds-between-progress-updates</a><br />
					</td>
					<!--<td>double</td> -->
						<td>10.0</td>
					<td>Output traversal statistics every time this many seconds elapse</td>
				</tr>
				<tr>
					<td><a href="#--sequence-dictionary">--sequence-dictionary</a><br />
					</td>
					<!--<td>GATKPath</td> -->
						<td></td>
					<td>Use the given sequence dictionary as the master/canonical sequence dictionary.  Must be a .dict file.</td>
				</tr>
				<tr>
					<td><a href="#--tmp-dir">--tmp-dir</a><br />
					</td>
					<!--<td>GATKPath</td> -->
						<td></td>
					<td>Temp directory to use.</td>
				</tr>
				<tr>
					<td><a href="#--use-jdk-deflater">--use-jdk-deflater</a><br />
								&nbsp;<em>-jdk-deflater</em>
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>Whether to use the JdkDeflater (as opposed to IntelDeflater)</td>
				</tr>
				<tr>
					<td><a href="#--use-jdk-inflater">--use-jdk-inflater</a><br />
								&nbsp;<em>-jdk-inflater</em>
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>Whether to use the JdkInflater (as opposed to IntelInflater)</td>
				</tr>
				<tr>
					<td><a href="#--verbosity">--verbosity</a><br />
					</td>
					<!--<td>LogLevel</td> -->
						<td>INFO</td>
					<td>Control verbosity of logging.</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Arguments</th>
			</tr>
				<tr>
					<td><a href="#--min-num-bases-for-segment-funcotation">--min-num-bases-for-segment-funcotation</a><br />
					</td>
					<!--<td>int</td> -->
						<td>150</td>
					<td>The minimum number of bases for a variant to be annotated as a segment.  Recommended to be changed only for use with FuncotateSegments.  Defaults to 150</td>
				</tr>
				<tr>
					<td><a href="#--showHidden">--showHidden</a><br />
					</td>
					<!--<td>boolean</td> -->
						<td>false</td>
					<td>display hidden arguments</td>
				</tr>
					</tbody>
				</table>

					<h3>Argument details</h3>
					<p>Arguments in this list are specific to this tool. Keep in mind that other arguments are available that are shared with other tools (e.g. command-line GATK arguments); see Inherited arguments above.</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--add-output-sam-program-record">--add-output-sam-program-record </a>
			 / <small>-add-output-sam-program-record</small>
		</h3>
		<p class="args">
			<b>If true, adds a PG tag to created SAM/BAM/CRAM files.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">true</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--add-output-vcf-command-line">--add-output-vcf-command-line </a>
			 / <small>-add-output-vcf-command-line</small>
		</h3>
		<p class="args">
			<b>If true, adds a command line header line to created VCF files.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">true</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--annotation-default">--annotation-default </a>
			
		</h3>
		<p class="args">
			<b>Annotations to include in all annotated variants if the annotation is not specified in the data sources (in the format <ANNOTATION>:<VALUE>).  This will add the specified annotation to every annotated variant if it is not already present.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--annotation-override">--annotation-override </a>
			
		</h3>
		<p class="args">
			<b>Override values for annotations (in the format <ANNOTATION>:<VALUE>).  Replaces existing annotations of the given name with given values.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--arguments_file">--arguments_file </a>
			
		</h3>
		<p class="args">
			<b>read one or more arguments files and add them to the command line</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[File]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--cloud-index-prefetch-buffer">--cloud-index-prefetch-buffer </a>
			 / <small>-CIPB</small>
		</h3>
		<p class="args">
			<b>Size of the cloud-only prefetch buffer (in MB; 0 to disable). Defaults to cloudPrefetchBuffer if unset.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">-1</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--cloud-prefetch-buffer">--cloud-prefetch-buffer </a>
			 / <small>-CPB</small>
		</h3>
		<p class="args">
			<b>Size of the cloud-only prefetch buffer (in MB; 0 to disable).</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">40</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--create-output-bam-index">--create-output-bam-index </a>
			 / <small>-OBI</small>
		</h3>
		<p class="args">
			<b>If true, create a BAM/CRAM index when writing a coordinate-sorted BAM/CRAM file.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">true</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--create-output-bam-md5">--create-output-bam-md5 </a>
			 / <small>-OBM</small>
		</h3>
		<p class="args">
			<b>If true, create a MD5 digest for any BAM/SAM/CRAM file created</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--create-output-variant-index">--create-output-variant-index </a>
			 / <small>-OVI</small>
		</h3>
		<p class="args">
			<b>If true, create a VCF index when writing a coordinate-sorted VCF file.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">true</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--create-output-variant-md5">--create-output-variant-md5 </a>
			 / <small>-OVM</small>
		</h3>
		<p class="args">
			<b>If true, create a a MD5 digest any VCF file created.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--data-sources-path">--data-sources-path </a>
			
		</h3>
		<p class="args">
			<b>The path to a data source folder for Funcotator.  May be specified more than once to handle multiple data source folders.</b><br />
			
		</p>
		<p>
				<span class="badge badge-important">R</span>
		<span class="label label-info ">List[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--disable-bam-index-caching">--disable-bam-index-caching </a>
			 / <small>-DBIC</small>
		</h3>
		<p class="args">
			<b>If true, don't cache bam indexes, this will reduce memory requirements but may harm performance if many intervals are specified.  Caching is automatically disabled if there are no intervals specified.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--disable-read-filter">--disable-read-filter </a>
			 / <small>-DF</small>
		</h3>
		<p class="args">
			<b>Read filters to be disabled before analysis</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--disable-sequence-dictionary-validation">--disable-sequence-dictionary-validation </a>
			 / <small>-disable-sequence-dictionary-validation</small>
		</h3>
		<p class="args">
			<b>If specified, do not check the sequence dictionaries from our inputs for compatibility. Use at your own risk!</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--disable-tool-default-read-filters">--disable-tool-default-read-filters </a>
			 / <small>-disable-tool-default-read-filters</small>
		</h3>
		<p class="args">
			<b>Disable all tool default read filters (WARNING: many tools will not function correctly without their default read filters on)</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--exclude-field">--exclude-field </a>
			
		</h3>
		<p class="args">
			<b>Fields that should not be rendered in the final output.  Only exact name matches will be excluded.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">Set[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--exclude-intervals">--exclude-intervals </a>
			 / <small>-XL</small>
		</h3>
		<p class="args">
			<b>One or more genomic intervals to exclude from processing</b><br />
			Use this argument to exclude certain parts of the genome from the analysis (like -L, but the opposite).
 This argument can be specified multiple times. You can use samtools-style intervals either explicitly on the
 command line (e.g. -XL 1 or -XL 1:100-200) or by loading in a file containing a list of intervals
 (e.g. -XL myFile.intervals).
		</p>
		<p>
		<span class="label label-info ">List[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--five-prime-flank-size">--five-prime-flank-size </a>
			
		</h3>
		<p class="args">
			<b>Variants within this many bases of the 5' end of a transcript (and not overlapping any part of the transcript itself) will be annotated as being in the 5' flanking region of that transcript</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">5000</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--gatk-config-file">--gatk-config-file </a>
			
		</h3>
		<p class="args">
			<b>A configuration file to use with the GATK.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">String</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--gcs-max-retries">--gcs-max-retries </a>
			 / <small>-gcs-retries</small>
		</h3>
		<p class="args">
			<b>If the GCS bucket channel errors out, how many times it will attempt to re-initiate the connection</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">20</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--gcs-project-for-requester-pays">--gcs-project-for-requester-pays </a>
			
		</h3>
		<p class="args">
			<b>Project to bill when accessing "requester pays" buckets. If unset, these buckets cannot be accessed.  User must have storage.buckets.get permission on the bucket being accessed.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">String</span>
			&nbsp;<span class="label">""</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--help">--help </a>
			 / <small>-h</small>
		</h3>
		<p class="args">
			<b>display the help message</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--input">--input </a>
			 / <small>-I</small>
		</h3>
		<p class="args">
			<b>BAM/SAM/CRAM file containing reads</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[GATKPath]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--interval-exclusion-padding">--interval-exclusion-padding </a>
			 / <small>-ixp</small>
		</h3>
		<p class="args">
			<b>Amount of padding (in bp) to add to each interval you are excluding.</b><br />
			Use this to add padding to the intervals specified using -XL. For example, '-XL 1:100' with a
 padding value of 20 would turn into '-XL 1:80-120'. This is typically used to add padding around targets when
 analyzing exomes.
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">0</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--interval-merging-rule">--interval-merging-rule </a>
			 / <small>-imr</small>
		</h3>
		<p class="args">
			<b>Interval merging rule for abutting intervals</b><br />
			By default, the program merges abutting intervals (i.e. intervals that are directly side-by-side but do not
 actually overlap) into a single continuous interval. However you can change this behavior if you want them to be
 treated as separate intervals instead.
		</p>
			<p>
				The --interval-merging-rule argument is an enumerated type (IntervalMergingRule), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">ALL</dt>
					<dd class="enum"></dd>
					<dt class="enum">OVERLAPPING_ONLY</dt>
					<dd class="enum"></dd>
			</dl>
			</p>
		<p>
		<span class="label label-info ">IntervalMergingRule</span>
			&nbsp;<span class="label">ALL</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--interval-padding">--interval-padding </a>
			 / <small>-ip</small>
		</h3>
		<p class="args">
			<b>Amount of padding (in bp) to add to each interval you are including.</b><br />
			Use this to add padding to the intervals specified using -L. For example, '-L 1:100' with a
 padding value of 20 would turn into '-L 1:80-120'. This is typically used to add padding around targets when
 analyzing exomes.
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">0</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--interval-set-rule">--interval-set-rule </a>
			 / <small>-isr</small>
		</h3>
		<p class="args">
			<b>Set merging approach to use for combining interval inputs</b><br />
			By default, the program will take the UNION of all intervals specified using -L and/or -XL. However, you can
 change this setting for -L, for example if you want to take the INTERSECTION of the sets instead. E.g. to
 perform the analysis only on chromosome 1 exomes, you could specify -L exomes.intervals -L 1 --interval-set-rule
 INTERSECTION. However, it is not possible to modify the merging approach for intervals passed using -XL (they will
 always be merged using UNION).

 Note that if you specify both -L and -XL, the -XL interval set will be subtracted from the -L interval set.
		</p>
			<p>
				The --interval-set-rule argument is an enumerated type (IntervalSetRule), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">UNION</dt>
					<dd class="enum">Take the union of all intervals</dd>
					<dt class="enum">INTERSECTION</dt>
					<dd class="enum">Take the intersection of intervals (the subset that overlaps all intervals specified)</dd>
			</dl>
			</p>
		<p>
		<span class="label label-info ">IntervalSetRule</span>
			&nbsp;<span class="label">UNION</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--intervals">--intervals </a>
			 / <small>-L</small>
		</h3>
		<p class="args">
			<b>One or more genomic intervals over which to operate</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--lenient">--lenient </a>
			 / <small>-LE</small>
		</h3>
		<p class="args">
			<b>Lenient processing of VCF files</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--lookahead-cache-bp">--lookahead-cache-bp </a>
			
		</h3>
		<p class="args">
			<b>Number of base-pairs to cache when querying variants.  Can be overridden in individual data source configuration files.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">100000</span>
			&nbsp;<span class="label label-warning">[ [ 0</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max-variants-per-shard">--max-variants-per-shard </a>
			
		</h3>
		<p class="args">
			<b>If non-zero, partitions VCF output into shards, each containing up to the given number of records.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">0</span>
			&nbsp;<span class="label label-warning">[ [ 0</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--min-num-bases-for-segment-funcotation">--min-num-bases-for-segment-funcotation </a>
			
		</h3>
		<p class="args">
			<b>The minimum number of bases for a variant to be annotated as a segment.  Recommended to be changed only for use with FuncotateSegments.  Defaults to 150</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">150</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--output">--output </a>
			 / <small>-O</small>
		</h3>
		<p class="args">
			<b>Output file to which annotated variants should be written.</b><br />
			
		</p>
		<p>
				<span class="badge badge-important">R</span>
		<span class="label label-info ">File</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--output-file-format">--output-file-format </a>
			
		</h3>
		<p class="args">
			<b>The output file format.  Either VCF, MAF, or SEG.  Please note that MAF output for germline use case VCFs is unsupported.  SEG will generate two output files: a simple tsv and a gene list.</b><br />
			
		</p>
			<p>
				The --output-file-format argument is an enumerated type (OutputFormatType), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">VCF</dt>
					<dd class="enum"></dd>
					<dt class="enum">MAF</dt>
					<dd class="enum"></dd>
					<dt class="enum">SEG</dt>
					<dd class="enum"></dd>
			</dl>
			</p>
		<p>
				<span class="badge badge-important">R</span>
		<span class="label label-info ">OutputFormatType</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--QUIET">--QUIET </a>
			
		</h3>
		<p class="args">
			<b>Whether to suppress job-summary info on System.err.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">Boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--read-filter">--read-filter </a>
			 / <small>-RF</small>
		</h3>
		<p class="args">
			<b>Read filters to be applied before analysis</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--read-index">--read-index </a>
			 / <small>-read-index</small>
		</h3>
		<p class="args">
			<b>Indices to use for the read inputs. If specified, an index must be provided for every read input and in the same order as the read inputs. If this argument is not specified, the path to the index for each input will be inferred automatically.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[GATKPath]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--read-validation-stringency">--read-validation-stringency </a>
			 / <small>-VS</small>
		</h3>
		<p class="args">
			<b>Validation stringency for all SAM/BAM/CRAM/SRA files read by this program.  The default stringency value SILENT can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) do not otherwise need to be decoded.</b><br />
			
		</p>
			<p>
				The --read-validation-stringency argument is an enumerated type (ValidationStringency), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">STRICT</dt>
					<dd class="enum"></dd>
					<dt class="enum">LENIENT</dt>
					<dd class="enum"></dd>
					<dt class="enum">SILENT</dt>
					<dd class="enum"></dd>
			</dl>
			</p>
		<p>
		<span class="label label-info ">ValidationStringency</span>
			&nbsp;<span class="label">SILENT</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--reannotate-vcf">--reannotate-vcf </a>
			
		</h3>
		<p class="args">
			<b>When input VCF has already been annotated, still annotate again.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--ref-version">--ref-version </a>
			
		</h3>
		<p class="args">
			<b>The version of the Human Genome reference to use (e.g. hg19, hg38, etc.).  This will correspond to a sub-folder of each data source corresponding to that data source for the given reference.</b><br />
			
		</p>
		<p>
				<span class="badge badge-important">R</span>
		<span class="label label-info ">String</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--reference">--reference </a>
			 / <small>-R</small>
		</h3>
		<p class="args">
			<b>Reference sequence file</b><br />
			
		</p>
		<p>
				<span class="badge badge-important">R</span>
		<span class="label label-info ">GATKPath</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--remove-filtered-variants">--remove-filtered-variants </a>
			
		</h3>
		<p class="args">
			<b>Ignore/drop variants that have been filtered in the input.  These variants will not appear in the output file.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--seconds-between-progress-updates">--seconds-between-progress-updates </a>
			 / <small>-seconds-between-progress-updates</small>
		</h3>
		<p class="args">
			<b>Output traversal statistics every time this many seconds elapse</b><br />
			
		</p>
		<p>
		<span class="label label-info ">double</span>
			&nbsp;<span class="label">10.0</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--sequence-dictionary">--sequence-dictionary </a>
			 / <small>-sequence-dictionary</small>
		</h3>
		<p class="args">
			<b>Use the given sequence dictionary as the master/canonical sequence dictionary.  Must be a .dict file.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">GATKPath</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--showHidden">--showHidden </a>
			 / <small>-showHidden</small>
		</h3>
		<p class="args">
			<b>display hidden arguments</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--sites-only-vcf-output">--sites-only-vcf-output </a>
			
		</h3>
		<p class="args">
			<b>If true, don't emit genotype fields when writing vcf file output.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--three-prime-flank-size">--three-prime-flank-size </a>
			
		</h3>
		<p class="args">
			<b>Variants within this many bases of the 3' end of a transcript (and not overlapping any part of the transcript itself) will be annotated as being in the 3' flanking region of that transcript</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">0</span>
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--tmp-dir">--tmp-dir </a>
			
		</h3>
		<p class="args">
			<b>Temp directory to use.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">GATKPath</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--transcript-list">--transcript-list </a>
			
		</h3>
		<p class="args">
			<b>File to use as a list of transcripts (one transcript ID per line, version numbers are ignored) OR A set of transcript IDs to use for annotation to override selected transcript.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">Set[String]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--transcript-selection-mode">--transcript-selection-mode </a>
			
		</h3>
		<p class="args">
			<b>Method of detailed transcript selection.  This will select the transcript for detailed annotation (CANONICAL, ALL, or BEST_EFFECT).</b><br />
			
		</p>
			<p>
				The --transcript-selection-mode argument is an enumerated type (TranscriptSelectionMode), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">BEST_EFFECT</dt>
					<dd class="enum">BEST_EFFECT

 Select a transcript to be reported with details with priority on effect according to the folowing list of selection criteria:

 Choose the transcript that is on the custom list specified by the user. If no list was specified, treat as if no transcripts were on the list (tie).
 In case of tie, choose the transcript that yields the variant classification highest on the variant classification rank list (see below).
 If still a tie, choose the transcript with highest level of curation. Note that this means lower number is better for level (see below).
 If still a tie, choose the transcript with the best appris annotation (see below).
 If still a tie, choose the transcript with the longest transcript sequence length.
 If still a tie, choose the first transcript, alphabetically.

 Levels of Curation:
     1 (verified loci)
     2 manually annotated loci
     3 automatically annotated loci

 Variant Classification Scores (See {@link org.broadinstitute.hellbender.tools.funcotator.dataSources.gencode.GencodeFuncotation.VariantClassification} as well):
      De_novo_Start_OutOfFrame    0
      Nonsense_Mutation           0
      Nonstop_Mutation            0
      Missense_Mutation           1
      De_novo_Start_InFrame       1
      In_Frame_Del                1
      In_Frame_Ins                1
      Frame_Shift_Del             2
      Frame_Shift_Ins             2
      Frame_Shift_Sub             2
      Start_Codon_SNP             3
      Start_Codon_Del             3
      Start_Codon_Ins             3
      Start_Codon_DNP             3
      Start_Codon_TNP             3
      Start_Codon_ONP             3
      Stop_Codon_SNP              3
      Stop_Codon_Del              3
      Stop_Codon_Ins              3
      Stop_Codon_DNP              3
      Stop_Codon_TNP              3
      Stop_Codon_ONP              3
      Splice_Site                 4
      Splice_Site_SNP             4
      Splice_Site_Del             4
      Splice_Site_Ins             4
      Splice_Site_DNP             4
      Splice_Site_TNP             4
      Splice_Site_ONP             4
      Splice_Site                 4
      miRNA                       4
      Silent                      5
      3UTR                        6
      5UTR                        6
      Intron                      7
      5Flank                      8
      3Flank                      8
      Non-coding_Transcript       9
      IGR	                        20
      TX-REF-MISMATCH             100

 APPRIS Ranks (http://appris.bioinfo.cnio.es/):

      appris_principal
      appris_candidate_highest_score
      appris_candidate_longest_ccds
      appris_candidate_ccds
      appris_candidate_longest_seq
      appris_candidate_longest
      appris_candidate
      no appris tag present</dd>
					<dt class="enum">CANONICAL</dt>
					<dd class="enum">CANONICAL

 Select a transcript to be reported with details with priority on canonical order according to the folowing list of selection criteria:

 Choose the transcript that is on the custom list specified by the user. If no list was specified, treat as if all transcripts were on the list (tie).
 In case of tie, choose the transcript with highest level of curation. Note that this means lower number is better for level (see below).
 If still a tie, choose the transcript that yields the variant classification highest on the variant classification rank list (see below).
 If still a tie, choose the transcript with the best appris annotation (see below).
 If still a tie, choose the transcript with the longest transcript sequence length.
 If still a tie, choose the first transcript, alphabetically.

 Levels of Curation:
     1 (verified loci)
     2 manually annotated loci
     3 automatically annotated loci

 Variant Classification Scores (See {@link GencodeFuncotation.VariantClassification} as well):
      De_novo_Start_OutOfFrame    0
      Nonsense_Mutation           0
      Nonstop_Mutation            0
      Missense_Mutation           1
      De_novo_Start_InFrame       1
      In_Frame_Del                1
      In_Frame_Ins                1
      Frame_Shift_Del             2
      Frame_Shift_Ins             2
      Frame_Shift_Sub             2
      Start_Codon_SNP             3
      Start_Codon_Del             3
      Start_Codon_Ins             3
      Start_Codon_DNP             3
      Start_Codon_TNP             3
      Start_Codon_ONP             3
      Stop_Codon_SNP              3
      Stop_Codon_Del              3
      Stop_Codon_Ins              3
      Stop_Codon_DNP              3
      Stop_Codon_TNP              3
      Stop_Codon_ONP              3
      Splice_Site                 4
      Splice_Site_SNP             4
      Splice_Site_Del             4
      Splice_Site_Ins             4
      Splice_Site_DNP             4
      Splice_Site_TNP             4
      Splice_Site_ONP             4
      Splice_Site                 4
      miRNA                       4
      Silent                      5
      3UTR                        6
      5UTR                        6
      Intron                      7
      5Flank                      8
      3Flank                      8
      Non-coding_Transcript       9
      IGR	                        20
      TX-REF-MISMATCH             100

 APPRIS Ranks (http://appris.bioinfo.cnio.es/):

      appris_principal
      appris_candidate_highest_score
      appris_candidate_longest_ccds
      appris_candidate_ccds
      appris_candidate_longest_seq
      appris_candidate_longest
      appris_candidate
      no appris tag present</dd>
					<dt class="enum">ALL</dt>
					<dd class="enum">Same as CANONICAL, but indicates that no transcripts should be dropped.  Render all overlapping transcripts.</dd>
			</dl>
			</p>
		<p>
		<span class="label label-info ">TranscriptSelectionMode</span>
			&nbsp;<span class="label">CANONICAL</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--use-jdk-deflater">--use-jdk-deflater </a>
			 / <small>-jdk-deflater</small>
		</h3>
		<p class="args">
			<b>Whether to use the JdkDeflater (as opposed to IntelDeflater)</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--use-jdk-inflater">--use-jdk-inflater </a>
			 / <small>-jdk-inflater</small>
		</h3>
		<p class="args">
			<b>Whether to use the JdkInflater (as opposed to IntelInflater)</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--variant">--variant </a>
			 / <small>-V</small>
		</h3>
		<p class="args">
			<b>A VCF file containing variants</b><br />
			
		</p>
		<p>
				<span class="badge badge-important">R</span>
		<span class="label label-info ">GATKPath</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--verbosity">--verbosity </a>
			 / <small>-verbosity</small>
		</h3>
		<p class="args">
			<b>Control verbosity of logging.</b><br />
			
		</p>
			<p>
				The --verbosity argument is an enumerated type (LogLevel), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">ERROR</dt>
					<dd class="enum"></dd>
					<dt class="enum">WARNING</dt>
					<dd class="enum"></dd>
					<dt class="enum">INFO</dt>
					<dd class="enum"></dd>
					<dt class="enum">DEBUG</dt>
					<dd class="enum"></dd>
			</dl>
			</p>
		<p>
		<span class="label label-info ">LogLevel</span>
			&nbsp;<span class="label">INFO</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--version">--version </a>
			
		</h3>
		<p class="args">
			<b>display the version number for this tool</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>

        <hr>
        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
        <hr>
        <p class="see-also">See also 
        	<a href="https://software.broadinstitute.org/gatk/documentation/">General Documentation</a> |
        	<a class="hide_me_php" href="index.html">Tool Docs Index</a> <a class="hide_me_html" href="index">Tool Documentation Index</a> |
        	<a href="https://gatkforums.broadinstitute.org/gatk/">Support Forum</a>
        </p>

        <p class="version">GATK version 4.2.4.0 built at Wed, 15 Dec 2021 12:49:57 -0500.
        </p>

</div></div>

	<?php printFooter($module); ?>